<!DOCTYPE html>
<html lang="en" dir="auto">

<head><meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>Binding and sequestration of poison frog alkaloids by a plasma globulin | Landingpage-in-a-Box</title>
<meta name="keywords" content="RNAseq, AlphaFold, map[FOS:Biological sciences], GCMS, LCMS, Molecular docking, Proteomics">
<meta name="description" content="Abstract Alkaloids are important bioactive molecules throughout the natural world, and in many animals, they serve as a source of chemical defense against predation. Dendrobatid poison frogs bioaccumulate alkaloids from their diet to make themselves toxic or unpalatable to predators. Despite the proposed roles of plasma proteins as mediators of alkaloid trafficking and bioavailability, the responsible proteins have not been identified. We use chemical approaches to show that a ~50 kDa plasma protein is the principal alkaloid binding molecule in blood from poison frogs.">
<meta name="author" content="Alvarez-Buylla, Aurora, O&#39;Connell, Lauren">
<link rel="canonical" href="https://datapurifier.github.io/landingpage-in-a-box/10.5061/dryad.mkkwh7143/">
<link crossorigin="anonymous" href="/landingpage-in-a-box/assets/css/stylesheet.4652fda8867d1835187bccecf539b89d6bb1fa82083e7fc03287396443458ee0.css" integrity="sha256-RlL9qIZ9GDUYe8zs9Tm4nWux&#43;oIIPn/AMoc5ZENFjuA=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/landingpage-in-a-box/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://datapurifier.github.io/landingpage-in-a-box/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://datapurifier.github.io/landingpage-in-a-box/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://datapurifier.github.io/landingpage-in-a-box/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://datapurifier.github.io/landingpage-in-a-box/apple-touch-icon.png">
<link rel="mask-icon" href="https://datapurifier.github.io/landingpage-in-a-box/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="Binding and sequestration of poison frog alkaloids by a plasma globulin" />
<meta property="og:description" content="Abstract Alkaloids are important bioactive molecules throughout the natural world, and in many animals, they serve as a source of chemical defense against predation. Dendrobatid poison frogs bioaccumulate alkaloids from their diet to make themselves toxic or unpalatable to predators. Despite the proposed roles of plasma proteins as mediators of alkaloid trafficking and bioavailability, the responsible proteins have not been identified. We use chemical approaches to show that a ~50 kDa plasma protein is the principal alkaloid binding molecule in blood from poison frogs." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://datapurifier.github.io/landingpage-in-a-box/10.5061/dryad.mkkwh7143/" /><meta property="article:section" content="10.5061" />



<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Binding and sequestration of poison frog alkaloids by a plasma globulin"/>
<meta name="twitter:description" content="Abstract Alkaloids are important bioactive molecules throughout the natural world, and in many animals, they serve as a source of chemical defense against predation. Dendrobatid poison frogs bioaccumulate alkaloids from their diet to make themselves toxic or unpalatable to predators. Despite the proposed roles of plasma proteins as mediators of alkaloid trafficking and bioavailability, the responsible proteins have not been identified. We use chemical approaches to show that a ~50 kDa plasma protein is the principal alkaloid binding molecule in blood from poison frogs."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [
    {
      "@type": "ListItem",
      "position":  1 ,
      "name": "10.5061",
      "item": "https://datapurifier.github.io/landingpage-in-a-box/10.5061/"
    }, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Binding and sequestration of poison frog alkaloids by a plasma globulin",
      "item": "https://datapurifier.github.io/landingpage-in-a-box/10.5061/dryad.mkkwh7143/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "Binding and sequestration of poison frog alkaloids by a plasma globulin",
  "name": "Binding and sequestration of poison frog alkaloids by a plasma globulin",
  "description": "Abstract Alkaloids are important bioactive molecules throughout the natural world, and in many animals, they serve as a source of chemical defense against predation. Dendrobatid poison frogs bioaccumulate alkaloids from their diet to make themselves toxic or unpalatable to predators. Despite the proposed roles of plasma proteins as mediators of alkaloid trafficking and bioavailability, the responsible proteins have not been identified. We use chemical approaches to show that a ~50 kDa plasma protein is the principal alkaloid binding molecule in blood from poison frogs.",
  "keywords": [
    "RNAseq", "AlphaFold", {"FOS":"Biological sciences"}, "GCMS", "LCMS", "Molecular docking", "Proteomics"
  ],
  "articleBody": "Abstract Alkaloids are important bioactive molecules throughout the natural world, and in many animals, they serve as a source of chemical defense against predation. Dendrobatid poison frogs bioaccumulate alkaloids from their diet to make themselves toxic or unpalatable to predators. Despite the proposed roles of plasma proteins as mediators of alkaloid trafficking and bioavailability, the responsible proteins have not been identified. We use chemical approaches to show that a ~50 kDa plasma protein is the principal alkaloid binding molecule in blood from poison frogs. Proteomic and biochemical studies establish this plasma protein to be liver-derived alkaloid-binding globulin (ABG) that is a member of the serine-protease inhibitor (serpin) family. In addition to alkaloid binding activity, ABG sequesters and regulates the bioavailability of “free” plasma alkaloids in vitro. Unexpectedly, ABG is not related to saxiphilin or albumin but instead exhibits sequence and structural homology to mammalian hormone carriers and amphibian biliverdin binding proteins. Alkaloid-binding globulin (ABG) represents a new small molecule binding functionality in serpin proteins, a novel mechanism of plasma alkaloid transport in poison frogs, and more broadly points towards serpins acting as tunable scaffolds for small molecule binding and transport across different organisms.\nMethods Animal usage All animal procedures were approved by the Institutional Animal Care and Use Committee at Stanford (protocol #34153). Topical benzocaine was used for anesthesia prior to the euthanasia of all animals. Laboratory-bred animals were purchased from Understory Enterprises (Ontario, Canada) or Josh’s Frogs (Michigan, USA) depending on the species. Animals were either euthanized for plasma collection upon arrival, or housed in 183 inch glass terraria, and fed a diet of non-toxic Drosophila melanogaster until euthanasia. Plasma and tissues from a total of 62 animals were used for this study, consisting of 32 lab-bred animals and 30 field-collected animals that are described below. Plasma collections Lab-bred, and therefore non-toxic, poison frogs were anesthetized with topical application of 10% benzocaine on the ventral skin, and euthanized with cervical translocation. Blood was collected directly from the cervical cut using a heparinized capillary tube (22-362-566, Fisher Scientific, Waltham, MA) and deposited into a lithium-heparin-coated microvette tube (20.1282.100, Sarstedt, Nümbrecht, Germany). Blood was spun down for 10 minutes at 5000 rpm at 4°C on a benchtop centrifuge, and the top layer which contains the plasma was removed and pipetted into a microcentrifuge tube. This was stored at -80°C until it was used for experiments. UV crosslinking and competition using alkaloid-like photoprobe Photocrosslinking methods follow methods outlined in Kim et al., 2020 [1]. Plasma or purified protein was thawed on ice. The total reaction volume was 50 uL and all experiments were performed in a clear 96-well plate. For plasma, 5 μL of undiluted plasma was mixed with 40 μL of PBS for each reaction. For purified O. sylvatica protein, 10 μg of protein was diluted into PBS per reaction to a volume of 45 μL. For E. tricolor and D. tinctorius protein, 60 μg and 35 μg were used, respectively; a higher amount of protein was used because no photocrosslinking was detected at 10 μg for these proteins. To this, either 2.5 μL of DMSO was added as vehicle control, or 2.5 μL of competitor compound dissolved in DMSO was added at a final concentration of 100 μM for plasma competition experiments unless indicated otherwise, or 1 mM for purified protein experiments unless indicated otherwise. The competitor compounds were: custom synthesized pumiliotoxin 251D (PTX, PepTech, Burlington, MA), decahydroquinoline (DHQ, 125741, Sigma-Aldrich, St. Louis, MO), epibatidine (epi, E1145, Sigma-Aldrich), a histrionicotoxin-like compound (HTX, ENAH2C55884A-50MG, Sigma-Aldrich), indolizidine (indol, ATE24584802-100MG, Sigma-Aldrich), nicotine (nic, N3876-100ML, Sigma-Aldrich), cortisol (cort, H0888-1G, Sigma-Aldrich). The “toxin mixture” (TM) used as a competitor in Figure 6 was made by taking 20μL of each of the skin alkaloid extracts from wild frogs described below, evaporating it under gentle nitrogen gas flow, and resuspending in 100μL of DMSO. This was followed by addition of 2.5 μL of photoprobe (Z2866906198, Enamine, Kyiv, Ukraine) dissolved in DMSO on ice, for a final photoprobe concentration of 5 μM in plasma experiments and 100 μM in purified protein experiments. This was incubated on ice for 10 minutes, and then UV crosslinked (Stratalinker UV 1800 Crosslinker, Stratagene, La Jolla, CA) for 5 minutes on ice. TAMRA visualization of crosslinked proteins was done by adding 3 μl TBTA (stock solution: 1.7 mM in 4:1 v/v DMSO:tert-Butanol; H66485-03, Fisher), 1 μl Copper (II) Sulfate (stock solution: 50 mM in water; BP346-500, Fisher), 1 μl Tris (2-carboxyethyl) phosphine hydrochloride (freshly prepared, stock solution: 50 mM; J60316-06, Fisher), and 1 μl TAMRA-N3 (stock solution: 1.25 mM in DMSO; T10182, Fisher), incubating at room temperature for 1 hour, and quenching the reaction by boiling with 4x SDS loading buffer. This was run on a Nupage 4-12% Bis-Tris protein gel (NP0323BOX, Invitrogen, Waltham, MA) and the in-gel fluorescence of the gel was visualized using a LI-COR Odyssey imaging system (LI-COR Biosciences, Lincoln, Nebraska) at 600 nm for an exposure time of 30 seconds. After imaging the TAMRA signal, the same gel was coomassie stained (InstantBlue, ISB1L, Abcam, Cambridge, UK) and visualized the same way at 700 nm. For proteomic identification of competed proteins, plasma samples were pooled from five different individuals and were crosslinked with either no photoprobe and equivalent amounts of DMSO, 5 μM photoprobe and DMSO, or 5 μM photoprobe and 100 μM PTX as described above. Each condition was set up as 24 individual reactions and pooled after crosslinking. To attach a biotin handle, 3 μl TBTA, 1 ul CuSO4, 1 μl TCEP, and 1.14 μl Biotin-N3 (stock solution: 9.67 mM in DMSO; 1265, Click Chemistry Tools, Scottsdale, AZ), were added for each reaction and this was incubated at room temperature for 1 hour, rotating. After incubation, each condition was run through a 3kDa MWCO centrifuge filter twice (UFC800324, Amicon, Millipore-Sigma, Burlington, MA) to dilute excess Biotin-N3 until reaching a 900x dilution. Pulldown of biotinylated photoprobe-protein complexes was achieved with a magnetic bead strep pulldown following the protocol outlined in Wei et al., 2021[2]. The pre and post-pulldown samples were run on a gel for a streptavidin blot (Figure 2A) and silver stain. After verifying pulldown efficacy, samples were run on SDS-PAGE gels in two replicates (one lane each) for each condition. Gels were fixed in 50:50 water:MeOH with 10% Acetic Acid for 1-2 hours. For the first replicate, the gel was run for a short period, and an approximately one-centimeter squared portion containing the whole lane for each condition was excised and fixed. For the second replicate, the gel was run completely and the proteins between 39 kDa and 64 kDa were excised and fixed using the ladder as a size reference. Sections were chopped into 1 mm pieces under sterile conditions and stored at 4°C in 100μL of water with 1% acetic acid until processed for proteomics. Proteomic identification of pulled down proteins across conditions For proteomics analyses, SDS-PAGE gel slices approximately 1 cm in length were prepared for proteolytic digestion. Each fixed gel slice was diced into 1 mm cubes under sterile conditions, and then rinsed with 50 mM ammonium bicarbonate to remove residual acidification from the fixing process. Following rinsing, the gels were incubated in 80% acetonitrile in water for five minutes; the solvent was removed and then the gel pieces were incubated with 10 mM DTT dissolved in water at room temperature for 20 minutes. Following reduction, alkylation was performed using 30 mM acrylamide for 30 minutes at room temperature to cap free reduced cysteines. Proteolysis was performed using trypsin/lysC (Promega, Madison, WI) in 50 mM ammonium bicarbonate overnight at 37°C. Resulting samples were spun to pellet gel fragments prior to extraction of the peptides present in the supernatant. The resulting peptides were dried by speed vac before dissolution in a reconstitution buffer (2% acetonitrile with 0.1% formic acid), with an estimated 1 μg on-column used for subsequent LC-MS/MS analysis. The liquid chromatography mass spectrometry experiment was performed using an Orbitrap Eclipse Tribrid mass spectrometer RRID:022212 (Thermo Scientific, San Jose, CA) with liquid chromatography using an Acquity M-Class UPLC (Waters Corporation, Milford, MA). A flow rate of 300 nL/min was used, where mobile phase A was 0.2% formic acid in water and mobile phase B was 0.2% formic acid in acetonitrile. Analytical columns were prepared in-house with an I.D. of 100 microns pulled to a nanospray emitter using a P2000 laser puller (Sutter Instrument, Novato, CA). The column was packed using C18 reprosil Pur 1.8 micron stationary phase (Dr. Maisch) to an approximate length of 25 cm. Peptides were directly injected onto the analytical column using a 80-minute gradient (2%–45% B, followed by a high-B wash). The mass spectrometer was operated in a data-dependent fashion using CID fragmentation in the ion trap for MS/MS spectra generation. For data analysis, the .RAW data files were processed using Byonic v4.1.5 (Protein Metrics, Cupertino, CA) to identify peptides and infer proteins based on a proteomic reference created with the O. sylvatica genome annotation. Proteolysis with Trypsin/LysC was assumed to be specific with up to 2 missed proteolytic cleavages. Precursor mass accuracies were held within 12 ppm, and 0.4 Da for MS/MS fragments in the ion trap. Cysteine modified with propionamide were set as fixed modifications in the search, and other common modifications (e.g. oxidation of methionine) were also included. Proteins were held to a false discovery rate of 1%, using standard reverse-decoy technique [3]. Identification of ABG proteins in different species and sequence confirmation To identify potential ABG proteins in other species, we used the OsABG protein sequence identified in the proteomics as the query and searched against blast databases created from the Allobates femoralis genome, and Epipedobates tricolor, Dendrobates tinctorius, and Mantella aurantiaca transcriptomes. The top hit from each blast search was used as the most probable ABG gene from those species. To ensure that the sequences did not contain sequencing or alignment errors, the gene from O. sylvatica, D. tinctorius, and E. tricolor was amplified using PCR and sequence confirmed with Sanger sequencing. Total RNA was extracted from flash-frozen liver tissue from three lab-bred, non-toxic, individuals from each species using the Monarch total RNA Miniprep Kit (T2010S, New England Biolabs, Ipswich, MA) following manufacturer instructions. This was used to create cDNA using the SuperScript III First-Strand Synthesis kit (18080-400, Invitrogen), following manufacturer instructions with an oligo(dT)20 primer. This was used for a PCR using Phusion High Fidelity DNA polymerase (F-530, Thermo Scientific) and the primers and cycling conditions described below. PCRs were analyzed using a 1% agarose gel for presence of a single band, cleaned up (NucleoSpin Gel and PCR cleanup, 740609.50, Takara Bio, Shiga, Japan), and transformed into pENTR vectors using a D-TOPO kit (45-0218, Invitrogen). Plasmids containing the ABG sequences from each individual were then mini-prepped (27106X4, Qiagen, Hilden, Germany) and Sanger sequenced with M13F and M13R primers (Azenta Life Sciences, South San Francisco, California). Sequences were aligned using Benchling (Benchling Inc, San Francisco, California) software, with MAFFT used for DNA alignments and Clustal Omega used for protein alignments. species fwd primer rev primer Tm O. sylvatica CACCATGAAACTTTTCGTCTACCTGTGTTTCAGC CTATTTTGTTGGGTCTACTATTCTTCCGCTG 68°C D. tinctorius CACCATGAAGCTTTTCGTCTTCCTATGTTTCAGCC CTATTTTGTTGGGTTTATTATTTTTCCATTCAAAATATCG 66°C E. tricolor CACCATGAAGCTTTTCATCTTCCTGTGTTTGAGCC CTATTTTGTTGGGTCTATTATTCTTCCGGAGAAAAC 68°C Cycling conditions: 98°C for 30 seconds, [98°C for 10 seconds, Tm for 30 seconds, 72°C for 2 minutes] x 34 cycles, 72°C for 10 minutes Protein structure prediction and molecular docking analyses The OsABG protein folding was predicted using the amino acid sequence, edited for point mutations found across all three individuals used for sequence verification, and the AlphaFold Google Colab notebook (https://colab.research.google.com/github/deepmind/alphafold/blob/main/notebooks/AlphaFold.ipynb)[4]. The predicted structure is provided in the supplementary information. The default AlphaFold parameters were used. Molecular docking was performed using the UCSF Chimera software (https://www.cgl.ucsf.edu/chimera/)[5], using AutoDock Vina [6,7] with the three-dimensional structure of PTX 251D (Pubchem CID 6440480). The whole protein was used as the search space with the default search parameters (5 binding modes, exhaustiveness of search of 8, and a maximum energy difference of 3 kcal/mol). The docking result with the highest predicted affinity was used and is included in the supplementary files. Protein structures and docking were visualized using PyMol for publication-quality images. Recombinant protein expression and binding assays Recombinant ABG proteins were expressed by Kemp Proteins (Maryland, USA) through their custom insect cell protein expression and purification services. The reagents and vectors used are proprietary to Kemp Proteins, however, the general expression and purification details are as follows. The verified protein sequences described above or the point mutations (Figure 4) were codon-optimized for SF9 insect expression, and a 10xHIS tag was added to the C-terminal end. For OsABG, a 1 L expression was performed, for all other sequences (other species and mutants) a 50 mL expression was used. For the 1 L expression, a multiplicity of infection of one was used for the p1 baculovirus, and the supernatant was collected after 72 hours. To this, 5 mL of Qiagen Ni-NTA resin washed and equilibrated in Buffer A (20 mM Sodium Phosphate, 300 mM NaCl, pH=7.8) was added and it was mixed overnight at 4°C. Afterwards, it was packed in a 5 mL Bio-Scale column and washed with 3 column volumes (CV) of Buffer A, followed by washing with 5% Buffer B (20 mM Sodium Phosphate, 300 mM NaCl, 500 mM Imidazole, pH=7.8) for 5 CV. Protein was eluted with a linear gradient from 5-60% over 25 CV, and 6 mL fractions were collected throughout. All of the fractions containing protein were pooled and concentrated to 1 mg/mL using an Amicon centrifugal filter with a 10 kDa MWCO, the buffer was exchanged to PBS, it was filtered through a 0.2 um filter, aliquoted, and frozen at -80°C. Protein expression and purification resulted in a clear band by western blot (Figure S4A) and a clean doublet pattern by coomassie (Figure S4B) closely resembling that seen in the plasma crosslinking results (Figure 1C) in both reduced and non-reduced conditions. For the 50mL expression of DtABG, EtABG, and mutant OsABG proteins, a 10% ratio of p1 virus to media was used and the supernatant was collected after 72 hours, to which 1 mL of Qiagen Ni-NTA resin washed and equilibrated in Buffer A (20 mM Sodium Phosphate, 300 mM NaCl, pH=7.4) was added. This was mixed overnight at 4°C and then packed into a 1 mL Bio-Scale column, washed with 3 CV of Buffer A, washed with 5 CV of 5% Buffer B (20 mM Sodium Phosphate, 300 mM NaCl, 500 mM imidazole, pH=7.4), and eluted with 5 CV of 50% Buffer B. Fractions containing protein were buffer exchanged into PBS, and the final concentrations were approximately 0.2 mg/mL, with varying final volumes. Protein expression and purification resulted in a clear band by western blot (Figure S4C,E,G,I,K), and a clean doublet by coomassie (Figure S4D,F,H,J,L) in both reduced and non-reduced conditions. Determination of dissociation constant using Microscale Thermophoresis (MST) To determine the binding affinity of OsABG for PTX, we used Microscale Thermophoresis (MST) to determine the dissociation constant (KD). To do this, we used the Monolith system (Nanotemper Technologies, München, Germany). Purified OsABG protein was labeled using the protein labeling kit Red-NHS 2nd generation (MO-L011, Nanotemper) which dyes primary lysine residues in the protein. The kit was used following manufacturer instructions, however, a 1.5x excess of dye was used instead of 3x as this was found to better achieve a degree of labeling of ~0.5. To remove aggregates during the assay, PBS-Tween was used for protein labeling and all dilutions. The labeled protein was centrifuged for 10 minutes at 20,000g on a benchtop centrifuge, and the supernatant was taken to further remove any aggregation. The concentration was measured prior to calculating and setting up dilution series. A final concentration of 10nM OsABG was used, and a 16-tube 2x serial dilution series of PTX 251D was made with the highest concentration being 1000uM. The concentration of DMSO was maintained consistent across the dilution series. The labeled OsABG was incubated for 10 minutes prior to loading into capillaries, and three biological replicates were pipetted and run separately. The Monolith premium capillaries (MO-K025, Nanotemper) were used, the MST power was set to Medium, and the excitation power was set to auto-detect. The three replicates were compiled and plotted together using GraphPad Prism (GraphPad Software, San Diego, California), and a dissociation model was fit to the data to calculate the KD. The raw data is included in the supplementary information (will be included with full submission). Determination of free versus bound alkaloids Solutions with 4 uM of OsABG protein, 4 uM of either PTX 251D or nicotine, and a final volume of 100 uL were made and incubated for one hour at room temperature. This was transferred to a 3 kDa MWCO centrifugal filter (UFC500396, Amicon) and spun at max speed on a benchtop centrifuge at 4C for 45 minutes. The top and bottom fractions were brought up to 100 uL with ultrapure water and transferred to new tubes, where 300 uL of 2:1 Acetonitrile:Methanol was added, after which they were vortexed and centrifuged at max speed on a benchtop centrifuge at 4C for 10 minutes. The supernatant was transferred to autosampler vials for quantitation of the amount of alkaloid in each fraction with mass spectrometry. Each condition was run in triplicate. Samples were analyzed using an Agilent Quadrupole time-of-flight LC-MS instrument, with MS analysis performed by electrospray ionization (ESI) in positive mode. Metabolites were separated with an Eclipse Plus C18 column (Agilent 959961-902) with normal phase chromatography. Mobile phases were: buffer A (water with 0.1% formic acid) and buffer B (90% acetonitrile, 10% water with 0.1% formic acid). The flow rate was maintained constant at 0.7 mL/min throughout the LC protocol. The LC gradient elution was set as follows: starting at 5% B held till 0.51 minutes, linear gradient from 5 to 25% B in 1.5 minutes, linear gradient from 25 to 50% B in 23 minutes, linear gradient from 50 to 95% B in 30 seconds, 95% B held for 2 minutes, linear gradient from 95 to 5% B in 1 minute, and 5% B held for 1.5 minutes to equilibrate the column to the initial conditions. The total run time was 30 minutes and the injection volume was 10 uL. Data was analyzed using the Agilent MassHunter software; the extracted ion chromatograms for PTX were searched using the exact mass M+1 of 252.2333, and nicotine was searched using the exact mass M+1 of 163.123, with a tolerance of a symmetric +/- 100 ppm. Extracted ion chromatograms were smoothed once before automatically integrating to get the abundance values. Abundance values were used to calculate the fractions above and below the filter for each replicate, which were then plotted with GraphPad. All raw data is provided as mzXML files here. Field Collections of Oophaga sylvatica The frog samples used in this paper are the same as those used for the project described in Moskowitz et al., 2022 [8]. For each location, 10 O. sylvatica individuals were collected under collection permit 0013-18 IC-FAU-DNB/MA issued by the Ministerio del Ambiente de Ecuador, between the hours of 7:00–18:00 during early May to early June 2019. All individuals were euthanized the same day as collection. Prior to euthanizing, frogs were sexed, weighed, and the snout-vent length was measured. Orajel (10% benzocaine) was used as an anesthetic prior to cervical dislocation. Once euthanized, frogs were immediately dissected and the liver, intestines, and half of the dorsal skin were stored in RNAlater in cryotubes at room temperature. The other half of the dorsal skin was placed in methanol in glass tubes at room temperature. Once back in the lab, all tissues were stored at -20°C until further processing. All tissues were transported to the United States under CITES permits 19EC000036/VS, 19EC000037/VS, 19EC000038/VS. Alkaloid extraction, detection, and analysis All the following steps were performed under a hood. Skins were taken out of methanol with forceps and weighed. From the methanol that the skin was stored in, 1 mL was taken and syringe filtered through a 0.45μ PTFE filter (44504-NP, Thermo Scientific) into the new glass vial with a PTFE cap (60940A-2, Fisher) filled with 25 μL of 1 μg/μL (-)-Nicotine (N3876-100ML, Sigma Aldrich), for a total of 25 μg of added nicotine. Tubes were capped and vortexed, and stored at -80°C for 24 hours, during which proteins and lipids should precipitate. After 24 hours, tubes were taken out of the -80°C and quickly syringe filtered through a 0.45μ PTFE filter again into a new glass vial. A 100 μL aliquot was added to a GC-MS autosampler vial, and remaining solution was stored in the original capped vial at -80°C. GC-MS analysis was performed on a Shimadzu GCMS-QP2020 instrument with a Shimadzu 30m x 0.25 mmID SH-Rxi-5Sil MS column closely following the protocol outlined in Saporito et al., 2010 [9]. In brief, GC separation of alkaloids was achieved using a temperature program from 100 to 280°C at a rate of 10°C per minute with He as the carrier gas (flow rate: 1 mL/min). This was followed by a 2-minute hold and additional ramp to 320°C at a rate of 10°C/minute for column protection reasons, and no alkaloids appeared during this part of the method. Compounds were analyzed with electron impact-mass spectrometry (EI-MS). The GC-MS data files were exported as CDF files and the Global Natural Products Social Network (GNPS) was used to perform the deconvolution and library searching against the AMDIS (NIST) database to identify all compounds (https://gnps.ucsd.edu) [10]. For deconvolution (identification of peaks and abundance estimates) the default parameters were used, for the library search the precursor ion mass tolerance was set to 20000 Da and the MS/MS fragment ion tolerance to 0.5 Da. The resulting dataset was filtered to keep only compounds that matched to our spiked-in nicotine standard, alkaloids previously found in poison frogs from the Daly 2005 database [11], or compounds with the same base ring structure and R groups as the classes defined in Daly 2005. All GC-MS data as CDF files are available through the GNPS public data repository (accessing information will be added for full submission). Once the feature table from the GNPS deconvolution was filtered to only include only poison frog alkaloids and nicotine, the abundance values were normalized by dividing by the nicotine standard and skin weight. This filtered and normalized feature table was used for all further analyses and visualizations. All steps were carried out with R version 4.0.4, and code is included in supplementary data. RNA extraction and library preparation RNA extraction followed the Trizol (15596018, Thermo Fisher) RNA isolation protocol outlined in Caty et al. 2019 [12] according to the manufacturer’s instructions, and with sample randomization to avoid batch effects. RNA quality was measured on a Agilent Tapestation RNA screentape (Agilent, Santa Clara, CA), and quantified using a Qubit Broad Range RNA kit (Q10210, Invitrogen). In the liver and intestines, samples with RIN scores greater than 5 were kept, RNA was normalized to the same Qubit concentration, and mRNA was isolated and library prepped using the NEB Directional RNA sequencing kit (E7765L, New England Biolabs) with the PolyA purification bundle (E7490L, New England Biolabs) and 96 Unique Dual Indices (E7765L, New England Biolabs). The skin RIN scores were much lower, signaling potential RNA degradation, ribosomal degradation was instead used to isolate mRNA. Following normalization within all skin RNA samples to the same Qubit concentration, we used the Zymo RiboFree Total RNA Library Prep kit (R3003-B, Zymo Research, Irvine, CA) following manufacturer instructions. After library prep for all tissues was complete library size was quantified with the Agilent Tapestation D1000 screentape, and concentration was measured with the Qubit dsDNA high sensitivity kit (Q33231, Invitrogen). All libraries within a tissue type were pooled to equimolar amounts and sequenced on two lanes of an Illumina NovaSeq (Illumina, San Diego, CA) machine to obtain 150 bp paired-end reads. RNA expression analysis and identification of O. sylvatica serpinA genes Analysis of RNA expression levels followed the protocol outlined by Payne et al., 2022 [13]. The Trim-galore! wrapper tool [14] was used to trim adapter sequences with cutadapt [15] and quality filter the reads (trim_galore –paired –phred33 –length 36 -q 30 –stringency 1 -e 0.001). All trimmed reads are available through the NCBI BioProject (accessing information will be added for full submission). Kallisto [16] was used to pseudoalign the reads to a reference created with the coding sequence of the annotated O. sylvatica genome. These abundances were combined into a matrix, and the trimmed-mean of M-values (TMM) normalized counts were used for all further analyses. Additional serpinA genes were found in the genome by searching for all genes annotated with “serpina” in the header, and by blasting the OsABG protein sequence against the genome (e-value \u003c 1e-60) and including any additional genes not annotated with “serpina.” Four sequences were removed because they were exact matches of the full gene (OopSylGTT00000004683), the N-terminal end (OopSylGTT00000004650, OopSylGTT00000004685), or the C-terminal (OopSylGTT00000004676) end sequence of another serpina gene, and therefore could be potential duplications caused by annotation or assembly errors. To create the protein tree (Figure 6D), ClustalW was used to align the sequences, a distance matrix was created using identity, and neighbor joining was used to construct the tree. The albumin gene was determined by blasting the protein sequences of Xenopus laevis albumin A (Uniprot #P08759), X. laevis albumin B (Uniprot #P14872), and the Asian toad Bombina maxima albumin (Uniprot #Q3T478) against the O. sylvatica genome. In all three cases, the top hit was the same (OopSylGTT00000003067), therefore this was assumed to be the most likely albumin candidate in the genome and was used to plot the TMM expression for comparison. All plots were created in R version 4.0.4, and all analysis and plotting code is available in the supplementary files (will be included with full submission). References 1. Kim JT, Jedrychowski MP, Wei W, Fernandez D, Fischer CR, Banik SM, et al. A Plasma Protein Network Regulates PM20D1 and N-Acyl Amino Acid Bioactivity. Cell Chem Biol. 2020;27: 1130–1139.e4. 2. Wei W, Riley NM, Lyu X, Bertozzi CR, Long JZ. Protocol for cell type-specific labeling, enrichment, and proteomic profiling of plasma proteins in mice. STAR Protoc. 2021;2: 101014. 3. Elias JE, Gygi SP. Target-decoy search strategy for increased confidence in large-scale protein identifications by mass spectrometry. Nat Methods. 2007;4: 207–214. 4. Jumper J, Evans R, Pritzel A, Green T, Figurnov M, Ronneberger O, et al. Highly accurate protein structure prediction with AlphaFold. Nature. 2021;596: 583–589. 5. Pettersen EF, Goddard TD, Huang CC, Couch GS, Greenblatt DM, Meng EC, et al. UCSF Chimera–a visualization system for exploratory research and analysis. J Comput Chem. 2004;25: 1605–1612. 6. Trott O, Olson AJ. AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading. J Comput Chem. 2010;31: 455–461. 7. Eberhardt J, Santos-Martins D, Tillack AF, Forli S. AutoDock Vina 1.2.0: New Docking Methods, Expanded Force Field, and Python Bindings. J Chem Inf Model. 2021;61: 3891–3898. 8. Moskowitz NA, Aurora-Alvarez-Buylla, Morrison CR, Chamba A, Rentería J, Tapia EE, et al. Poison frog diet and chemical defense are influenced by availability and selectivity for ants. doi:10.1101/2022.06.14.495949 9. Saporito RA, Donnelly MA, Madden AA, Garraffo HM, Spande TF. Sex-related differences in alkaloid chemical defenses of the dendrobatid frog Oophaga pumilio from Cayo Nancy, Bocas del Toro, Panama. J Nat Prod. 2010;73: 317–321. 10. Wang M, Carver JJ, Phelan VV, Sanchez LM, Garg N, Peng Y, et al. Sharing and community curation of mass spectrometry data with Global Natural Products Social Molecular Networking. Nat Biotechnol. 2016;34: 828–837. 11. Daly JW, Spande TF, Garraffo HM. Alkaloids from Amphibian Skin: A Tabulation of Over Eight-Hundred Compounds. J Nat Prod. 2005;68: 1556–1575. 12. Caty SN, Alvarez-Buylla A, Byrd GD, Vidoudez C, Roland AB, Tapia EE, et al. Molecular physiology of chemical defenses in a poison frog. J Exp Biol. 2019;222: jeb204149. 13. Payne C, Bovio R, Powell DL, Gunn TR, Banerjee SM, Grant V, et al. Genomic insights into variation in thermotolerance between hybridizing swordtail fishes. Mol Ecol. 2022. doi:10.1111/mec.16489 14. Krueger F, James F, Ewels P, Afyounian E, Schuster-Boeckler BF. TrimGalore: v0. 6.7-DOI via Zenodo. Zenodo; 2021. 15. Martin M. Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet.journal. 2011. p. 10. doi:10.14806/ej.17.1.200 16. Bray NL, Pimentel H, Melsted P, Pachter L. Near-optimal probabilistic RNA-seq quantification. Nat Biotechnol. 2016;34: 525–527.\nOther Text editor, Excel (can also be opened by Google Drive), Pymol (can also be opened with UCSF Chimera and others), R, Agilent MassHunter (can also be opened with MZmine 3 http://mzmine.github.io/)\nTechnicalInfo Binding and sequestration of poison frog alkaloids by a plasma globulin This dryad upload contains the raw proteomics and mass spectrometry data associated with the eLife article by Alvarez-Buylla et al., 2023. The processed versions of these data are included with the main article as “Source data” files. Also provided on dryad are the annotated versions of the A. femoralis and O. sylvatica genomes, as well as E. tricolor, D. tinctorius, and M. aurantiaca transcriptomes. ## Description of the data and file structure ### Annotated genomes Allobates femoralis coding regions: PO1440_Allobates_femoralis.transcript.fasta.gz Allobates femoralis proteome: PO1440_Allobates_femoralis.protein.fasta.gz Oophaga sylvatica coding regions: combined_with_CDS_BRAKER_appended_high_conf.fun.CDS.fasta Oophaga sylvatica proteome: combined_with_CDS_BRAKER_appended_high_conf.fun.pep ### Transcriptomes Epipedobates tricolor: etricolor_tox_trinity.fasta Dendrobates tinctorius: AAB_E3_dtinct_trinity.Trinity.cleanedcontigs.fasta Mantella aurantiaca: maurantiaca_trinity.Trinity.fasta ### Raw proteomics files (Figure 2) No photoprobe control replicate 1: no_photoprobe_R1.raw.zip No photoprobe control replicate 2: no_photoprobe_R2.raw.zip Photoprobe only replicate 1: photoprobe_R1.raw.zip Photoprobe only replicate 2: photoprobe_R2.raw.zip Photoprobe with PTX competitor replicate 1: photoprobe+PTX_R1.raw.zip Photoprobe with PTX competitor replicate 2: photoprobe+PTX_R2.raw.zip ### Raw LC-MS files (Figure 5) There are two files for each sample, one with “top” and one with “bottom” in the filename. These refer to the top and bottom fractions of the molecular weight cutoff filter experiment in Figure 5, and the quantities in both were used to calculate the “fraction bound” in Figure 5. The analyzed data is in the source data files of the manuscript. For each condition there are three replicates indicated by different numbers: PTX with no protein: 1_bottom.mzXML, 1_top.mzXML, 2_bottom.mzXML, 2_top.mzXML, 3_bottom.mzXML, 3_top.mzXML PTX with OsABG: 7_bottom.mzXML, 7_top.mzXML, 8_bottom.mzXML, 8_top.mzXML, 9_bottom.mzXML, 9_top.mzXML Nicotine with no protein: 13_bottom.mzXML,13_top.mzXML, 14_bottom.mzXML, 14_top.mzXML, 15_bottom.mzXML, 15_top.mzXML Nicotine with OsABG: 16_bottom.mzXML, 16_top.mzXML, 17_bottom.mzXML, 17_top.mzXML, 18_bottom.mzXML, 18_top.mzXML Standards and blanks: Blanks: 00_Blank1_C18.mzXML, 00_Blank2_C18.mzXML 1 uM of Nicotine and PTX: 1uM_STD.mzXML 10 uM of Nicotine and PTX: 10uM_STD.mzXML 100 nM of Nicotine and PTX: 100nM_STD.mzXML ### Raw GC-MS files (Figure 6) Figure6_rawGCMSfile.zip contains the raw GC-MS files from field-collected Oophaga sylvatica individuals. Each file represents one individual’s skin alkaloid profile. The data shown in the manuscript compiles the alkaloid classes across all of the individuals, and the identified and quantified peaks can be found in the source data files of the manuscript.\nAccess Points https://datadryad.org/stash/dataset/doi:10.5061/dryad.mkkwh7143\nRelated Identifiers IsCitedBy https://doi.org/10.1101/2022.11.22.517437 ",
  "wordCount" : "5099",
  "inLanguage": "en",
  "datePublished": "0001-01-01T00:00:00Z",
  "dateModified": "0001-01-01T00:00:00Z",
  "author":[{
    "@type": "Person",
    "name": "Alvarez-Buylla, Aurora"
  }, {
    "@type": "Person",
    "name": "O'Connell, Lauren"
  }],
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://datapurifier.github.io/landingpage-in-a-box/10.5061/dryad.mkkwh7143/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "Landingpage-in-a-Box",
    "logo": {
      "@type": "ImageObject",
      "url": "https://datapurifier.github.io/landingpage-in-a-box/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://datapurifier.github.io/landingpage-in-a-box/" accesskey="h" title="Landingpage-in-a-Box (Alt + H)">Landingpage-in-a-Box</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">

<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      Binding and sequestration of poison frog alkaloids by a plasma globulin
    </h1>
    <div class="post-meta">

</div>
  </header> 
    
    
      
        
            <div class="added-meta" style="display: flex; align-items: center;">
    Alvarez-Buylla, Aurora
    
        <a href="https://orcid.org/0000-0001-6256-0300">
            <img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon" style="width:16px;height:16px;">
        </a>
    
    
    
         (Stanford University)
    
</div>
        
            <div class="added-meta" style="display: flex; align-items: center;">
    O&#39;Connell, Lauren
    
    
    
         (Stanford University)
    
</div>
        
    <div class="added-meta">
    Published: 2022 on Dryad<br />


    <a href="https://doi.org/10.5061/dryad.mkkwh7143">
    <img src="https://img.shields.io/badge/DOI-10.5061/dryad.mkkwh7143-blue" alt="DOI">
    </a>
    </div>

  <div class="post-content"><h1 id="abstract">Abstract<a hidden class="anchor" aria-hidden="true" href="#abstract">#</a></h1>
<p>Alkaloids are important bioactive molecules throughout the natural world, and in many animals, they serve as a source of chemical defense against predation. Dendrobatid poison frogs bioaccumulate alkaloids from their diet to make themselves toxic or unpalatable to predators. Despite the proposed roles of plasma proteins as mediators of alkaloid trafficking and bioavailability, the responsible proteins have not been identified. We use chemical approaches to show that a ~50 kDa plasma protein is the principal alkaloid binding molecule in blood from poison frogs. Proteomic and biochemical studies establish this plasma protein to be liver-derived alkaloid-binding globulin (ABG) that is a member of the serine-protease inhibitor (serpin) family. In addition to alkaloid binding activity, ABG sequesters and regulates the bioavailability of “free” plasma alkaloids in vitro. Unexpectedly, ABG is not related to saxiphilin or albumin but instead exhibits sequence and structural homology to mammalian hormone carriers and amphibian biliverdin binding proteins. Alkaloid-binding globulin (ABG) represents a new small molecule binding functionality in serpin proteins, a novel mechanism of plasma alkaloid transport in poison frogs, and more broadly points towards serpins acting as tunable scaffolds for small molecule binding and transport across different organisms.</p>
<h2 id="methods">Methods<a hidden class="anchor" aria-hidden="true" href="#methods">#</a></h2>
<p>Animal usage All animal procedures were approved by the Institutional Animal Care and Use Committee at Stanford (protocol #34153). Topical benzocaine was used for anesthesia prior to the euthanasia of all animals. Laboratory-bred animals were purchased from Understory Enterprises (Ontario, Canada) or Josh’s Frogs (Michigan, USA) depending on the species. Animals were either euthanized for plasma collection upon arrival, or housed in 183 inch glass terraria, and fed a diet of non-toxic Drosophila melanogaster until euthanasia. Plasma and tissues from a total of 62 animals were used for this study, consisting of 32 lab-bred animals and 30 field-collected animals that are described below. Plasma collections Lab-bred, and therefore non-toxic, poison frogs were anesthetized with topical application of 10% benzocaine on the ventral skin, and euthanized with cervical translocation. Blood was collected directly from the cervical cut using a heparinized capillary tube (22-362-566, Fisher Scientific, Waltham, MA) and deposited into a lithium-heparin-coated microvette tube (20.1282.100, Sarstedt, Nümbrecht, Germany). Blood was spun down for 10 minutes at 5000 rpm at 4°C on a benchtop centrifuge, and the top layer which contains the plasma was removed and pipetted into a microcentrifuge tube. This was stored at -80°C until it was used for experiments. UV crosslinking and competition using alkaloid-like photoprobe Photocrosslinking methods follow methods outlined in Kim et al., 2020 [1]. Plasma or purified protein was thawed on ice. The total reaction volume was 50 uL and all experiments were performed in a clear 96-well plate. For plasma, 5 μL of undiluted plasma was mixed with 40 μL of PBS for each reaction. For purified O. sylvatica protein, 10 μg of protein was diluted into PBS per reaction to a volume of 45 μL. For E. tricolor and D. tinctorius protein, 60 μg and 35 μg were used, respectively; a higher amount of protein was used because no photocrosslinking was detected at 10 μg for these proteins. To this, either 2.5 μL of DMSO was added as vehicle control, or 2.5 μL of competitor compound dissolved in DMSO was added at a final concentration of 100 μM for plasma competition experiments unless indicated otherwise, or 1 mM for purified protein experiments unless indicated otherwise. The competitor compounds were: custom synthesized pumiliotoxin 251D (PTX, PepTech, Burlington, MA), decahydroquinoline (DHQ, 125741, Sigma-Aldrich, St. Louis, MO), epibatidine (epi, E1145, Sigma-Aldrich), a histrionicotoxin-like compound (HTX, ENAH2C55884A-50MG, Sigma-Aldrich), indolizidine (indol, ATE24584802-100MG, Sigma-Aldrich), nicotine (nic, N3876-100ML, Sigma-Aldrich), cortisol (cort, H0888-1G, Sigma-Aldrich). The “toxin mixture” (TM) used as a competitor in Figure 6 was made by taking 20μL of each of the skin alkaloid extracts from wild frogs described below, evaporating it under gentle nitrogen gas flow, and resuspending in 100μL of DMSO. This was followed by addition of 2.5 μL of photoprobe (Z2866906198, Enamine, Kyiv, Ukraine) dissolved in DMSO on ice, for a final photoprobe concentration of 5 μM in plasma experiments and 100 μM in purified protein experiments. This was incubated on ice for 10 minutes, and then UV crosslinked (Stratalinker UV 1800 Crosslinker, Stratagene, La Jolla, CA) for 5 minutes on ice. TAMRA visualization of crosslinked proteins was done by adding 3 μl TBTA (stock solution: 1.7 mM in 4:1 v/v DMSO:tert-Butanol; H66485-03, Fisher), 1 μl Copper (II) Sulfate (stock solution: 50 mM in water; BP346-500, Fisher), 1 μl Tris (2-carboxyethyl) phosphine hydrochloride (freshly prepared, stock solution: 50 mM; J60316-06, Fisher), and 1 μl TAMRA-N3 (stock solution: 1.25 mM in DMSO; T10182, Fisher), incubating at room temperature for 1 hour, and quenching the reaction by boiling with 4x SDS loading buffer. This was run on a Nupage 4-12% Bis-Tris protein gel (NP0323BOX, Invitrogen, Waltham, MA) and the in-gel fluorescence of the gel was visualized using a LI-COR Odyssey imaging system (LI-COR Biosciences, Lincoln, Nebraska) at 600 nm for an exposure time of 30 seconds. After imaging the TAMRA signal, the same gel was coomassie stained (InstantBlue, ISB1L, Abcam, Cambridge, UK) and visualized the same way at 700 nm. For proteomic identification of competed proteins, plasma samples were pooled from five different individuals and were crosslinked with either no photoprobe and equivalent amounts of DMSO, 5 μM photoprobe and DMSO, or 5 μM photoprobe and 100 μM PTX as described above. Each condition was set up as 24 individual reactions and pooled after crosslinking. To attach a biotin handle, 3 μl TBTA, 1 ul CuSO4, 1 μl TCEP, and 1.14 μl Biotin-N3 (stock solution: 9.67 mM in DMSO; 1265, Click Chemistry Tools, Scottsdale, AZ), were added for each reaction and this was incubated at room temperature for 1 hour, rotating. After incubation, each condition was run through a 3kDa MWCO centrifuge filter twice (UFC800324, Amicon, Millipore-Sigma, Burlington, MA) to dilute excess Biotin-N3 until reaching a 900x dilution. Pulldown of biotinylated photoprobe-protein complexes was achieved with a magnetic bead strep pulldown following the protocol outlined in Wei et al., 2021[2]. The pre and post-pulldown samples were run on a gel for a streptavidin blot (Figure 2A) and silver stain. After verifying pulldown efficacy, samples were run on SDS-PAGE gels in two replicates (one lane each) for each condition. Gels were fixed in 50:50 water:MeOH with 10% Acetic Acid for 1-2 hours. For the first replicate, the gel was run for a short period, and an approximately one-centimeter squared portion containing the whole lane for each condition was excised and fixed. For the second replicate, the gel was run completely and the proteins between 39 kDa and 64 kDa were excised and fixed using the ladder as a size reference. Sections were chopped into 1 mm pieces under sterile conditions and stored at 4°C in 100μL of water with 1% acetic acid until processed for proteomics. Proteomic identification of pulled down proteins across conditions For proteomics analyses, SDS-PAGE gel slices approximately 1 cm in length were prepared for proteolytic digestion. Each fixed gel slice was diced into 1 mm cubes under sterile conditions, and then rinsed with 50 mM ammonium bicarbonate to remove residual acidification from the fixing process. Following rinsing, the gels were incubated in 80% acetonitrile in water for five minutes; the solvent was removed and then the gel pieces were incubated with 10 mM DTT dissolved in water at room temperature for 20 minutes. Following reduction, alkylation was performed using 30 mM acrylamide for 30 minutes at room temperature to cap free reduced cysteines. Proteolysis was performed using trypsin/lysC (Promega, Madison, WI) in 50 mM ammonium bicarbonate overnight at 37°C. Resulting samples were spun to pellet gel fragments prior to extraction of the peptides present in the supernatant. The resulting peptides were dried by speed vac before dissolution in a reconstitution buffer (2% acetonitrile with 0.1% formic acid), with an estimated 1 μg on-column used for subsequent LC-MS/MS analysis. The liquid chromatography mass spectrometry experiment was performed using an Orbitrap Eclipse Tribrid mass spectrometer RRID:022212 (Thermo Scientific, San Jose, CA) with liquid chromatography using an Acquity M-Class UPLC (Waters Corporation, Milford, MA). A flow rate of 300 nL/min was used, where mobile phase A was 0.2% formic acid in water and mobile phase B was 0.2% formic acid in acetonitrile. Analytical columns were prepared in-house with an I.D. of 100 microns pulled to a nanospray emitter using a P2000 laser puller (Sutter Instrument, Novato, CA). The column was packed using C18 reprosil Pur 1.8 micron stationary phase (Dr. Maisch) to an approximate length of 25 cm. Peptides were directly injected onto the analytical column using a 80-minute gradient (2%–45% B, followed by a high-B wash). The mass spectrometer was operated in a data-dependent fashion using CID fragmentation in the ion trap for MS/MS spectra generation. For data analysis, the .RAW data files were processed using Byonic v4.1.5 (Protein Metrics, Cupertino, CA) to identify peptides and infer proteins based on a proteomic reference created with the O. sylvatica genome annotation. Proteolysis with Trypsin/LysC was assumed to be specific with up to 2 missed proteolytic cleavages. Precursor mass accuracies were held within 12 ppm, and 0.4 Da for MS/MS fragments in the ion trap. Cysteine modified with propionamide were set as fixed modifications in the search, and other common modifications (e.g. oxidation of methionine) were also included. Proteins were held to a false discovery rate of 1%, using standard reverse-decoy technique [3]. Identification of ABG proteins in different species and sequence confirmation To identify potential ABG proteins in other species, we used the OsABG protein sequence identified in the proteomics as the query and searched against blast databases created from the Allobates femoralis genome, and Epipedobates tricolor, Dendrobates tinctorius, and Mantella aurantiaca transcriptomes. The top hit from each blast search was used as the most probable ABG gene from those species. To ensure that the sequences did not contain sequencing or alignment errors, the gene from O. sylvatica, D. tinctorius, and E. tricolor was amplified using PCR and sequence confirmed with Sanger sequencing. Total RNA was extracted from flash-frozen liver tissue from three lab-bred, non-toxic, individuals from each species using the Monarch total RNA Miniprep Kit (T2010S, New England Biolabs, Ipswich, MA) following manufacturer instructions. This was used to create cDNA using the SuperScript III First-Strand Synthesis kit (18080-400, Invitrogen), following manufacturer instructions with an oligo(dT)20 primer. This was used for a PCR using Phusion High Fidelity DNA polymerase (F-530, Thermo Scientific) and the primers and cycling conditions described below. PCRs were analyzed using a 1% agarose gel for presence of a single band, cleaned up (NucleoSpin Gel and PCR cleanup, 740609.50, Takara Bio, Shiga, Japan), and transformed into pENTR vectors using a D-TOPO kit (45-0218, Invitrogen). Plasmids containing the ABG sequences from each individual were then mini-prepped (27106X4, Qiagen, Hilden, Germany) and Sanger sequenced with M13F and M13R primers (Azenta Life Sciences, South San Francisco, California). Sequences were aligned using Benchling (Benchling Inc, San Francisco, California) software, with MAFFT used for DNA alignments and Clustal Omega used for protein alignments. species fwd primer rev primer Tm O. sylvatica CACCATGAAACTTTTCGTCTACCTGTGTTTCAGC CTATTTTGTTGGGTCTACTATTCTTCCGCTG 68°C D. tinctorius CACCATGAAGCTTTTCGTCTTCCTATGTTTCAGCC CTATTTTGTTGGGTTTATTATTTTTCCATTCAAAATATCG 66°C E. tricolor CACCATGAAGCTTTTCATCTTCCTGTGTTTGAGCC CTATTTTGTTGGGTCTATTATTCTTCCGGAGAAAAC 68°C Cycling conditions: 98°C for 30 seconds, [98°C for 10 seconds, Tm for 30 seconds, 72°C for 2 minutes] x 34 cycles, 72°C for 10 minutes Protein structure prediction and molecular docking analyses The OsABG protein folding was predicted using the amino acid sequence, edited for point mutations found across all three individuals used for sequence verification, and the AlphaFold Google Colab notebook (<a href="https://colab.research.google.com/github/deepmind/alphafold/blob/main/notebooks/AlphaFold.ipynb)%5B4%5D">https://colab.research.google.com/github/deepmind/alphafold/blob/main/notebooks/AlphaFold.ipynb)[4]</a>. The predicted structure is provided in the supplementary information. The default AlphaFold parameters were used. Molecular docking was performed using the UCSF Chimera software (<a href="https://www.cgl.ucsf.edu/chimera/)%5B5%5D">https://www.cgl.ucsf.edu/chimera/)[5]</a>, using AutoDock Vina [6,7] with the three-dimensional structure of PTX 251D (Pubchem CID 6440480). The whole protein was used as the search space with the default search parameters (5 binding modes, exhaustiveness of search of 8, and a maximum energy difference of 3 kcal/mol). The docking result with the highest predicted affinity was used and is included in the supplementary files. Protein structures and docking were visualized using PyMol for publication-quality images. Recombinant protein expression and binding assays Recombinant ABG proteins were expressed by Kemp Proteins (Maryland, USA) through their custom insect cell protein expression and purification services. The reagents and vectors used are proprietary to Kemp Proteins, however, the general expression and purification details are as follows. The verified protein sequences described above or the point mutations (Figure 4) were codon-optimized for SF9 insect expression, and a 10xHIS tag was added to the C-terminal end. For OsABG, a 1 L expression was performed, for all other sequences (other species and mutants) a 50 mL expression was used. For the 1 L expression, a multiplicity of infection of one was used for the p1 baculovirus, and the supernatant was collected after 72 hours. To this, 5 mL of Qiagen Ni-NTA resin washed and equilibrated in Buffer A (20 mM Sodium Phosphate, 300 mM NaCl, pH=7.8) was added and it was mixed overnight at 4°C. Afterwards, it was packed in a 5 mL Bio-Scale column and washed with 3 column volumes (CV) of Buffer A, followed by washing with 5% Buffer B (20 mM Sodium Phosphate, 300 mM NaCl, 500 mM Imidazole, pH=7.8) for 5 CV. Protein was eluted with a linear gradient from 5-60% over 25 CV, and 6 mL fractions were collected throughout. All of the fractions containing protein were pooled and concentrated to 1 mg/mL using an Amicon centrifugal filter with a 10 kDa MWCO, the buffer was exchanged to PBS, it was filtered through a 0.2 um filter, aliquoted, and frozen at -80°C. Protein expression and purification resulted in a clear band by western blot (Figure S4A) and a clean doublet pattern by coomassie (Figure S4B) closely resembling that seen in the plasma crosslinking results (Figure 1C) in both reduced and non-reduced conditions. For the 50mL expression of DtABG, EtABG, and mutant OsABG proteins, a 10% ratio of p1 virus to media was used and the supernatant was collected after 72 hours, to which 1 mL of Qiagen Ni-NTA resin washed and equilibrated in Buffer A (20 mM Sodium Phosphate, 300 mM NaCl, pH=7.4) was added. This was mixed overnight at 4°C and then packed into a 1 mL Bio-Scale column, washed with 3 CV of Buffer A, washed with 5 CV of 5% Buffer B (20 mM Sodium Phosphate, 300 mM NaCl, 500 mM imidazole, pH=7.4), and eluted with 5 CV of 50% Buffer B. Fractions containing protein were buffer exchanged into PBS, and the final concentrations were approximately 0.2 mg/mL, with varying final volumes. Protein expression and purification resulted in a clear band by western blot (Figure S4C,E,G,I,K), and a clean doublet by coomassie (Figure S4D,F,H,J,L) in both reduced and non-reduced conditions. Determination of dissociation constant using Microscale Thermophoresis (MST) To determine the binding affinity of OsABG for PTX, we used Microscale Thermophoresis (MST) to determine the dissociation constant (KD). To do this, we used the Monolith system (Nanotemper Technologies, München, Germany). Purified OsABG protein was labeled using the protein labeling kit Red-NHS 2nd generation (MO-L011, Nanotemper) which dyes primary lysine residues in the protein. The kit was used following manufacturer instructions, however, a 1.5x excess of dye was used instead of 3x as this was found to better achieve a degree of labeling of ~0.5. To remove aggregates during the assay, PBS-Tween was used for protein labeling and all dilutions. The labeled protein was centrifuged for 10 minutes at 20,000g on a benchtop centrifuge, and the supernatant was taken to further remove any aggregation. The concentration was measured prior to calculating and setting up dilution series. A final concentration of 10nM OsABG was used, and a 16-tube 2x serial dilution series of PTX 251D was made with the highest concentration being 1000uM. The concentration of DMSO was maintained consistent across the dilution series. The labeled OsABG was incubated for 10 minutes prior to loading into capillaries, and three biological replicates were pipetted and run separately. The Monolith premium capillaries (MO-K025, Nanotemper) were used, the MST power was set to Medium, and the excitation power was set to auto-detect. The three replicates were compiled and plotted together using GraphPad Prism (GraphPad Software, San Diego, California), and a dissociation model was fit to the data to calculate the KD. The raw data is included in the supplementary information (will be included with full submission). Determination of free versus bound alkaloids Solutions with 4 uM of OsABG protein, 4 uM of either PTX 251D or nicotine, and a final volume of 100 uL were made and incubated for one hour at room temperature. This was transferred to a 3 kDa MWCO centrifugal filter (UFC500396, Amicon) and spun at max speed on a benchtop centrifuge at 4C for 45 minutes. The top and bottom fractions were brought up to 100 uL with ultrapure water and transferred to new tubes, where 300 uL of 2:1 Acetonitrile:Methanol was added, after which they were vortexed and centrifuged at max speed on a benchtop centrifuge at 4C for 10 minutes. The supernatant was transferred to autosampler vials for quantitation of the amount of alkaloid in each fraction with mass spectrometry. Each condition was run in triplicate. Samples were analyzed using an Agilent Quadrupole time-of-flight LC-MS instrument, with MS analysis performed by electrospray ionization (ESI) in positive mode. Metabolites were separated with an Eclipse Plus C18 column (Agilent 959961-902) with normal phase chromatography. Mobile phases were: buffer A (water with 0.1% formic acid) and buffer B (90% acetonitrile, 10% water with 0.1% formic acid). The flow rate was maintained constant at 0.7 mL/min throughout the LC protocol. The LC gradient elution was set as follows: starting at 5% B held till 0.51 minutes, linear gradient from 5 to 25% B in 1.5 minutes, linear gradient from 25 to 50% B in 23 minutes, linear gradient from 50 to 95% B in 30 seconds, 95% B held for 2 minutes, linear gradient from 95 to 5% B in 1 minute, and 5% B held for 1.5 minutes to equilibrate the column to the initial conditions. The total run time was 30 minutes and the injection volume was 10 uL. Data was analyzed using the Agilent MassHunter software; the extracted ion chromatograms for PTX were searched using the exact mass M+1 of 252.2333, and nicotine was searched using the exact mass M+1 of 163.123, with a tolerance of a symmetric +/- 100 ppm. Extracted ion chromatograms were smoothed once before automatically integrating to get the abundance values. Abundance values were used to calculate the fractions above and below the filter for each replicate, which were then plotted with GraphPad. All raw data is provided as mzXML files here. Field Collections of Oophaga sylvatica The frog samples used in this paper are the same as those used for the project described in Moskowitz et al., 2022 [8]. For each location, 10 O. sylvatica individuals were collected under collection permit 0013-18 IC-FAU-DNB/MA issued by the Ministerio del Ambiente de Ecuador, between the hours of 7:00–18:00 during early May to early June 2019. All individuals were euthanized the same day as collection. Prior to euthanizing, frogs were sexed, weighed, and the snout-vent length was measured. Orajel (10% benzocaine) was used as an anesthetic prior to cervical dislocation. Once euthanized, frogs were immediately dissected and the liver, intestines, and half of the dorsal skin were stored in RNAlater in cryotubes at room temperature. The other half of the dorsal skin was placed in methanol in glass tubes at room temperature. Once back in the lab, all tissues were stored at -20°C until further processing. All tissues were transported to the United States under CITES permits 19EC000036/VS, 19EC000037/VS, 19EC000038/VS. Alkaloid extraction, detection, and analysis All the following steps were performed under a hood. Skins were taken out of methanol with forceps and weighed. From the methanol that the skin was stored in, 1 mL was taken and syringe filtered through a 0.45μ PTFE filter (44504-NP, Thermo Scientific) into the new glass vial with a PTFE cap (60940A-2, Fisher) filled with 25 μL of 1 μg/μL (-)-Nicotine (N3876-100ML, Sigma Aldrich), for a total of 25 μg of added nicotine. Tubes were capped and vortexed, and stored at -80°C for 24 hours, during which proteins and lipids should precipitate. After 24 hours, tubes were taken out of the -80°C and quickly syringe filtered through a 0.45μ PTFE filter again into a new glass vial. A 100 μL aliquot was added to a GC-MS autosampler vial, and remaining solution was stored in the original capped vial at -80°C. GC-MS analysis was performed on a Shimadzu GCMS-QP2020 instrument with a Shimadzu 30m x 0.25 mmID SH-Rxi-5Sil MS column closely following the protocol outlined in Saporito et al., 2010 [9]. In brief, GC separation of alkaloids was achieved using a temperature program from 100 to 280°C at a rate of 10°C per minute with He as the carrier gas (flow rate: 1 mL/min). This was followed by a 2-minute hold and additional ramp to 320°C at a rate of 10°C/minute for column protection reasons, and no alkaloids appeared during this part of the method. Compounds were analyzed with electron impact-mass spectrometry (EI-MS). The GC-MS data files were exported as CDF files and the Global Natural Products Social Network (GNPS) was used to perform the deconvolution and library searching against the AMDIS (NIST) database to identify all compounds (<a href="https://gnps.ucsd.edu">https://gnps.ucsd.edu</a>) [10]. For deconvolution (identification of peaks and abundance estimates) the default parameters were used, for the library search the precursor ion mass tolerance was set to 20000 Da and the MS/MS fragment ion tolerance to 0.5 Da. The resulting dataset was filtered to keep only compounds that matched to our spiked-in nicotine standard, alkaloids previously found in poison frogs from the Daly 2005 database [11], or compounds with the same base ring structure and R groups as the classes defined in Daly 2005. All GC-MS data as CDF files are available through the GNPS public data repository (accessing information will be added for full submission). Once the feature table from the GNPS deconvolution was filtered to only include only poison frog alkaloids and nicotine, the abundance values were normalized by dividing by the nicotine standard and skin weight. This filtered and normalized feature table was used for all further analyses and visualizations. All steps were carried out with R version 4.0.4, and code is included in supplementary data. RNA extraction and library preparation RNA extraction followed the Trizol (15596018, Thermo Fisher) RNA isolation protocol outlined in Caty et al. 2019 [12] according to the manufacturer’s instructions, and with sample randomization to avoid batch effects. RNA quality was measured on a Agilent Tapestation RNA screentape (Agilent, Santa Clara, CA), and quantified using a Qubit Broad Range RNA kit (Q10210, Invitrogen). In the liver and intestines, samples with RIN scores greater than 5 were kept, RNA was normalized to the same Qubit concentration, and mRNA was isolated and library prepped using the NEB Directional RNA sequencing kit (E7765L, New England Biolabs) with the PolyA purification bundle (E7490L, New England Biolabs) and 96 Unique Dual Indices (E7765L, New England Biolabs). The skin RIN scores were much lower, signaling potential RNA degradation, ribosomal degradation was instead used to isolate mRNA. Following normalization within all skin RNA samples to the same Qubit concentration, we used the Zymo RiboFree Total RNA Library Prep kit (R3003-B, Zymo Research, Irvine, CA) following manufacturer instructions. After library prep for all tissues was complete library size was quantified with the Agilent Tapestation D1000 screentape, and concentration was measured with the Qubit dsDNA high sensitivity kit (Q33231, Invitrogen). All libraries within a tissue type were pooled to equimolar amounts and sequenced on two lanes of an Illumina NovaSeq (Illumina, San Diego, CA) machine to obtain 150 bp paired-end reads. RNA expression analysis and identification of O. sylvatica serpinA genes Analysis of RNA expression levels followed the protocol outlined by Payne et al., 2022 [13]. The Trim-galore! wrapper tool [14] was used to trim adapter sequences with cutadapt [15] and quality filter the reads (trim_galore &ndash;paired &ndash;phred33 &ndash;length 36 -q 30 &ndash;stringency 1 -e 0.001). All trimmed reads are available through the NCBI BioProject (accessing information will be added for full submission). Kallisto [16] was used to pseudoalign the reads to a reference created with the coding sequence of the annotated O. sylvatica genome. These abundances were combined into a matrix, and the trimmed-mean of M-values (TMM) normalized counts were used for all further analyses. Additional serpinA genes were found in the genome by searching for all genes annotated with “serpina” in the header, and by blasting the OsABG protein sequence against the genome (e-value &lt; 1e-60) and including any additional genes not annotated with “serpina.” Four sequences were removed because they were exact matches of the full gene (OopSylGTT00000004683), the N-terminal end (OopSylGTT00000004650, OopSylGTT00000004685), or the C-terminal (OopSylGTT00000004676) end sequence of another serpina gene, and therefore could be potential duplications caused by annotation or assembly errors. To create the protein tree (Figure 6D), ClustalW was used to align the sequences, a distance matrix was created using identity, and neighbor joining was used to construct the tree. The albumin gene was determined by blasting the protein sequences of Xenopus laevis albumin A (Uniprot #P08759), X. laevis albumin B (Uniprot #P14872), and the Asian toad Bombina maxima albumin (Uniprot #Q3T478) against the O. sylvatica genome. In all three cases, the top hit was the same (OopSylGTT00000003067), therefore this was assumed to be the most likely albumin candidate in the genome and was used to plot the TMM expression for comparison. All plots were created in R version 4.0.4, and all analysis and plotting code is available in the supplementary files (will be included with full submission). References 1. Kim JT, Jedrychowski MP, Wei W, Fernandez D, Fischer CR, Banik SM, et al. A Plasma Protein Network Regulates PM20D1 and N-Acyl Amino Acid Bioactivity. Cell Chem Biol. 2020;27: 1130–1139.e4. 2. Wei W, Riley NM, Lyu X, Bertozzi CR, Long JZ. Protocol for cell type-specific labeling, enrichment, and proteomic profiling of plasma proteins in mice. STAR Protoc. 2021;2: 101014. 3. Elias JE, Gygi SP. Target-decoy search strategy for increased confidence in large-scale protein identifications by mass spectrometry. Nat Methods. 2007;4: 207–214. 4. Jumper J, Evans R, Pritzel A, Green T, Figurnov M, Ronneberger O, et al. Highly accurate protein structure prediction with AlphaFold. Nature. 2021;596: 583–589. 5. Pettersen EF, Goddard TD, Huang CC, Couch GS, Greenblatt DM, Meng EC, et al. UCSF Chimera&ndash;a visualization system for exploratory research and analysis. J Comput Chem. 2004;25: 1605–1612. 6. Trott O, Olson AJ. AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading. J Comput Chem. 2010;31: 455–461. 7. Eberhardt J, Santos-Martins D, Tillack AF, Forli S. AutoDock Vina 1.2.0: New Docking Methods, Expanded Force Field, and Python Bindings. J Chem Inf Model. 2021;61: 3891–3898. 8. Moskowitz NA, Aurora-Alvarez-Buylla, Morrison CR, Chamba A, Rentería J, Tapia EE, et al. Poison frog diet and chemical defense are influenced by availability and selectivity for ants. doi:10.1101/2022.06.14.495949 9. Saporito RA, Donnelly MA, Madden AA, Garraffo HM, Spande TF. Sex-related differences in alkaloid chemical defenses of the dendrobatid frog Oophaga pumilio from Cayo Nancy, Bocas del Toro, Panama. J Nat Prod. 2010;73: 317–321. 10. Wang M, Carver JJ, Phelan VV, Sanchez LM, Garg N, Peng Y, et al. Sharing and community curation of mass spectrometry data with Global Natural Products Social Molecular Networking. Nat Biotechnol. 2016;34: 828–837. 11. Daly JW, Spande TF, Garraffo HM. Alkaloids from Amphibian Skin: A Tabulation of Over Eight-Hundred Compounds. J Nat Prod. 2005;68: 1556–1575. 12. Caty SN, Alvarez-Buylla A, Byrd GD, Vidoudez C, Roland AB, Tapia EE, et al. Molecular physiology of chemical defenses in a poison frog. J Exp Biol. 2019;222: jeb204149. 13. Payne C, Bovio R, Powell DL, Gunn TR, Banerjee SM, Grant V, et al. Genomic insights into variation in thermotolerance between hybridizing swordtail fishes. Mol Ecol. 2022. doi:10.1111/mec.16489 14. Krueger F, James F, Ewels P, Afyounian E, Schuster-Boeckler BF. TrimGalore: v0. 6.7-DOI via Zenodo. Zenodo; 2021. 15. Martin M. Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet.journal. 2011. p. 10. doi:10.14806/ej.17.1.200 16. Bray NL, Pimentel H, Melsted P, Pachter L. Near-optimal probabilistic RNA-seq quantification. Nat Biotechnol. 2016;34: 525–527.</p>
<h2 id="other">Other<a hidden class="anchor" aria-hidden="true" href="#other">#</a></h2>
<p>Text editor, Excel (can also be opened by Google Drive), Pymol (can also be opened with UCSF Chimera and others), R, Agilent MassHunter (can also be opened with MZmine 3 <a href="http://mzmine.github.io/">http://mzmine.github.io/</a>)</p>
<h2 id="technicalinfo">TechnicalInfo<a hidden class="anchor" aria-hidden="true" href="#technicalinfo">#</a></h2>
<p>Binding and sequestration of poison frog alkaloids by a plasma globulin This dryad upload contains the raw proteomics and mass spectrometry data associated with the eLife article by Alvarez-Buylla et al., 2023. The processed versions of these data are included with the main article as &ldquo;Source data&rdquo; files. Also provided on dryad are the annotated versions of the A. femoralis and O. sylvatica genomes, as well as E. tricolor, D. tinctorius, and M. aurantiaca transcriptomes. ## Description of the data and file structure ### <strong>Annotated genomes</strong> <em>Allobates femoralis</em> coding regions: <em><strong>PO1440_Allobates_femoralis.transcript.fasta.gz</strong></em> <em>Allobates femoralis</em> proteome: <em><strong>PO1440_Allobates_femoralis.protein.fasta.gz</strong></em> <em>Oophaga sylvatica</em> coding regions: <em><strong>combined_with_CDS_BRAKER_appended_high_conf.fun.CDS.fasta</strong></em> <em>Oophaga sylvatica</em> proteome: <em><strong>combined_with_CDS_BRAKER_appended_high_conf.fun.pep</strong></em> ### <strong>Transcriptomes</strong> <em>Epipedobates tricolor</em>: <em><strong>etricolor_tox_trinity.fasta</strong></em> <em>Dendrobates tinctorius</em>: <em><strong>AAB_E3_dtinct_trinity.Trinity.cleanedcontigs.fasta</strong></em> <em>Mantella aurantiaca</em>: <em><strong>maurantiaca_trinity.Trinity.fasta</strong></em> ### <strong>Raw proteomics files (Figure 2)</strong> No photoprobe control replicate 1: <em><strong>no_photoprobe_R1.raw.zip</strong></em> No photoprobe control replicate 2: <em><strong>no_photoprobe_R2.raw.zip</strong></em> Photoprobe only replicate 1: <em><strong>photoprobe_R1.raw.zip</strong></em> Photoprobe only replicate 2: <em><strong>photoprobe_R2.raw.zip</strong></em> Photoprobe with PTX competitor replicate 1: <em><strong>photoprobe+PTX_R1.raw.zip</strong></em> Photoprobe with PTX competitor replicate 2: <em><strong>photoprobe+PTX_R2.raw.zip</strong></em> ### <strong>Raw LC-MS files (Figure 5)</strong> There are two files for each sample, one with &ldquo;top&rdquo; and one with &ldquo;bottom&rdquo; in the filename. These refer to the top and bottom fractions of the molecular weight cutoff filter experiment in Figure 5, and the quantities in both were used to calculate the &ldquo;fraction bound&rdquo; in Figure 5. The analyzed data is in the source data files of the manuscript. <em>For each condition there are three replicates indicated by different numbers:</em> PTX with no protein: <em><strong>1_bottom.mzXML, 1_top.mzXML, 2_bottom.mzXML, 2_top.mzXML, 3_bottom.mzXML, 3_top.mzXML</strong></em> PTX with OsABG: <em><strong>7_bottom.mzXML, 7_top.mzXML, 8_bottom.mzXML, 8_top.mzXML, 9_bottom.mzXML, 9_top.mzXML</strong></em> Nicotine with no protein: <em><strong>13_bottom.mzXML,13_top.mzXML, 14_bottom.mzXML, 14_top.mzXML, 15_bottom.mzXML, 15_top.mzXML</strong></em> Nicotine with OsABG: <em><strong>16_bottom.mzXML, 16_top.mzXML, 17_bottom.mzXML, 17_top.mzXML, 18_bottom.mzXML, 18_top.mzXML</strong></em> <em>Standards and blanks:</em> Blanks: <em><strong>00_Blank1_C18.mzXML, 00_Blank2_C18.mzXML</strong></em> 1 uM of Nicotine and PTX: <em><strong>1uM_STD.mzXML</strong></em> 10 uM of Nicotine and PTX: <em><strong>10uM_STD.mzXML</strong></em> 100 nM of Nicotine and PTX: <em><strong>100nM_STD.mzXML</strong></em> ### <strong>Raw GC-MS files (Figure 6)</strong> <em><strong>Figure6_rawGCMSfile.zip</strong></em> contains the raw GC-MS files from field-collected <em>Oophaga sylvatica</em> individuals. Each file represents one individual&rsquo;s skin alkaloid profile. The data shown in the manuscript compiles the alkaloid classes across all of the individuals, and the identified and quantified peaks can be found in the source data files of the manuscript.</p>
<h1 id="access-points">Access Points<a hidden class="anchor" aria-hidden="true" href="#access-points">#</a></h1>
<p><a href="https://datadryad.org/stash/dataset/doi:10.5061/dryad.mkkwh7143">https://datadryad.org/stash/dataset/doi:10.5061/dryad.mkkwh7143</a></p>
<h1 id="related-identifiers">Related Identifiers<a hidden class="anchor" aria-hidden="true" href="#related-identifiers">#</a></h1>
<h2 id="iscitedby">IsCitedBy<a hidden class="anchor" aria-hidden="true" href="#iscitedby">#</a></h2>
<ul>
<li><a href="https://doi.org/10.1101/2022.11.22.517437">https://doi.org/10.1101/2022.11.22.517437</a></li>
</ul>


  </div>

  
    
    
    
      
    
  

  <footer class="post-footer">
    <ul class="post-tags">
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/rnaseq/">RNAseq</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/alphafold/">AlphaFold</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/gcms/">GCMS</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/lcms/">LCMS</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/molecular-docking/">Molecular docking</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/proteomics/">Proteomics</a></li>
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://datapurifier.github.io/landingpage-in-a-box/">Landingpage-in-a-Box</a></span>
    <span>
        Powered by
        <a href="https://gohugo.io/" rel="noopener noreferrer" target="_blank">Hugo</a> &
        <a href="https://github.com/adityatelange/hugo-PaperMod/" rel="noopener" target="_blank">PaperMod</a>
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
