<!DOCTYPE html>
<html lang="en" dir="auto">

<head><meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>QuCo: quartet-based co-estimation of species trees and gene trees | Landingpage-in-a-Box</title>
<meta name="keywords" content="map[FOS:Biological sciences], Co-Estimation Methods, Phylogenomics, Bayesian phylogenetics, species tree inference">
<meta name="description" content="Abstract Motivation: Phylogenomics faces a dilemma: on the one hand, the most accurate species and gene tree estimation methods are those that co-estimate them; on the other hand, these co-estimation methods do not scale to moderately large numbers of species. The summary-based methods, which first infer gene trees independently and then combine them, are much more scalable but are prone to gene tree estimation error, which is inevitable when inferring trees from limited-length data.">
<meta name="author" content="Mirarab, Siavash, Rabiee, Maryam">
<link rel="canonical" href="https://datapurifier.github.io/landingpage-in-a-box/10.6076/d1cp4r/">
<link crossorigin="anonymous" href="/landingpage-in-a-box/assets/css/stylesheet.4652fda8867d1835187bccecf539b89d6bb1fa82083e7fc03287396443458ee0.css" integrity="sha256-RlL9qIZ9GDUYe8zs9Tm4nWux&#43;oIIPn/AMoc5ZENFjuA=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/landingpage-in-a-box/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://datapurifier.github.io/landingpage-in-a-box/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://datapurifier.github.io/landingpage-in-a-box/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://datapurifier.github.io/landingpage-in-a-box/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://datapurifier.github.io/landingpage-in-a-box/apple-touch-icon.png">
<link rel="mask-icon" href="https://datapurifier.github.io/landingpage-in-a-box/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="QuCo: quartet-based co-estimation of species trees and gene trees" />
<meta property="og:description" content="Abstract Motivation: Phylogenomics faces a dilemma: on the one hand, the most accurate species and gene tree estimation methods are those that co-estimate them; on the other hand, these co-estimation methods do not scale to moderately large numbers of species. The summary-based methods, which first infer gene trees independently and then combine them, are much more scalable but are prone to gene tree estimation error, which is inevitable when inferring trees from limited-length data." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://datapurifier.github.io/landingpage-in-a-box/10.6076/d1cp4r/" /><meta property="article:section" content="10.6076" />



<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="QuCo: quartet-based co-estimation of species trees and gene trees"/>
<meta name="twitter:description" content="Abstract Motivation: Phylogenomics faces a dilemma: on the one hand, the most accurate species and gene tree estimation methods are those that co-estimate them; on the other hand, these co-estimation methods do not scale to moderately large numbers of species. The summary-based methods, which first infer gene trees independently and then combine them, are much more scalable but are prone to gene tree estimation error, which is inevitable when inferring trees from limited-length data."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [
    {
      "@type": "ListItem",
      "position":  1 ,
      "name": "10.6076",
      "item": "https://datapurifier.github.io/landingpage-in-a-box/10.6076/"
    }, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "QuCo: quartet-based co-estimation of species trees and gene trees",
      "item": "https://datapurifier.github.io/landingpage-in-a-box/10.6076/d1cp4r/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "QuCo: quartet-based co-estimation of species trees and gene trees",
  "name": "QuCo: quartet-based co-estimation of species trees and gene trees",
  "description": "Abstract Motivation: Phylogenomics faces a dilemma: on the one hand, the most accurate species and gene tree estimation methods are those that co-estimate them; on the other hand, these co-estimation methods do not scale to moderately large numbers of species. The summary-based methods, which first infer gene trees independently and then combine them, are much more scalable but are prone to gene tree estimation error, which is inevitable when inferring trees from limited-length data.",
  "keywords": [
    {"FOS":"Biological sciences"}, "Co-Estimation Methods", "Phylogenomics", "Bayesian phylogenetics", "species tree inference"
  ],
  "articleBody": "Abstract Motivation: Phylogenomics faces a dilemma: on the one hand, the most accurate species and gene tree estimation methods are those that co-estimate them; on the other hand, these co-estimation methods do not scale to moderately large numbers of species. The summary-based methods, which first infer gene trees independently and then combine them, are much more scalable but are prone to gene tree estimation error, which is inevitable when inferring trees from limited-length data. Gene tree estimation error is not just random noise and can create biases such as long-branch attraction. Results: We introduce a scalable likelihood-based approach to co-estimation under the multi-species coalescent model. The method, called quartet co-estimation (QuCo), takes as input independently inferred distributions over gene trees and computes the most likely species tree topology and internal branch length for each quartet, marginalizing over gene tree topologies and ignoring branch lengths by making several simplifying assumptions. It then updates the gene tree posterior probabilities based on the species tree. The focus on gene tree topologies and the heuristic division to quartets enables fast likelihood calculations. We benchmark our method with extensive simulations for quartet trees in zones known to produce biased species trees and further with larger trees. We also run QuCo on a biological dataset of bees. Our results show better accuracy than the summary-based approach ASTRAL run on estimated gene trees. Availability and implementation: QuCo is available on https://github.com/maryamrabiee/quco. Supplementary information Supplementary data are available at Bioinformatics online.\nMethods Data are simulated by us and provided here for reproducibility.\nTechnicalInfo Quco Dataset — Date belonging to the following paper: Rabiee, Maryam, and Siavash Mirarab. “QuCo: Quartet-Based Co-Estimation of Species Trees and Gene Trees.” Bioinformatics 38, no. Supplement1 (June 24, 2022): i413–21. . ## Description of the data and file structure There are several files: ### Quartet-simulation-sequences.tar.gz The main simulations presented in the paper, which involve Felsenstein’s zone quartets. Here, we provide the simulated sequences. Files are of the form: rep.[CU]d/R[long]l-[short]s/[rep]/seq[seqlength]/sequences.tar.gz where * [seqlength] is the sequence length and is either 1600, 800, 400, 200 * [rep] is the replicate number, which is between 1 and 20 * [short] is the length of the short branch, and varies between 0.01, 0.02, 0.04, 0.08. * [long] is the length of the long branch and varies between 0.1, 0.2, 0.3, and 0.4. * [CU] is the length of the internal branch length in coalescent units (CU) and varies between 0.1, 0.2, and 0.3. Each file includes the simulated sequences in the fasta format for all genes of each repliate. ### Anomaly-simulations-mrbayes-outputs.tar.gz This includes the results of the anomaly zone simulations, and specifically the output of MrBayes. For each of the 50 replicated simulations, we include: * [id]/mrbayes-outputs.tar.gz Inside each archive, we have MrBayes MCMC sample from 200 loci. The files are named as follows, where locus id is the name of the locus and we have results of four chains (runs 1–4). Each .t file includes the MCMC samples in nexus format, as generated by MrBayes. * seq600/[locus id]/[locus id].nex.run[1/2/3/4].t ### BiologicalDataset_rj_tree_distribution.tar.gz Biological dataset of Bossert et al. (2021) with 32 species and 1291 UCEs (). Here, for reproducabililty, we provide MrBayes tree distributions. For each of the 1291 loci, you can find the following files: * uce-[ucid].run[1/2].t: The MCM sample in nexus format from MrBayes, for chain (run) 1 or 2. ## Sharing/Access information The rest of the data is available on * GitLab ## Code/Software The simulated data are generated using Simphy\nAccess Points https://datadryad.org/stash/dataset/doi:10.6076/D1CP4R\nRelated Identifiers IsCitedBy https://doi.org/bioinformatics/btac265 IsDerivedFrom https://github.com/maryamrabiee/quco ",
  "wordCount" : "585",
  "inLanguage": "en",
  "datePublished": "0001-01-01T00:00:00Z",
  "dateModified": "0001-01-01T00:00:00Z",
  "author":[{
    "@type": "Person",
    "name": "Mirarab, Siavash"
  }, {
    "@type": "Person",
    "name": "Rabiee, Maryam"
  }],
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://datapurifier.github.io/landingpage-in-a-box/10.6076/d1cp4r/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "Landingpage-in-a-Box",
    "logo": {
      "@type": "ImageObject",
      "url": "https://datapurifier.github.io/landingpage-in-a-box/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://datapurifier.github.io/landingpage-in-a-box/" accesskey="h" title="Landingpage-in-a-Box (Alt + H)">Landingpage-in-a-Box</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">

<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      QuCo: quartet-based co-estimation of species trees and gene trees
    </h1>
    <div class="post-meta">

</div>
  </header> 
    
    
      
        
            <div class="added-meta" style="display: flex; align-items: center;">
    <a href="../../author/mirarab-siavash/" style="text-decoration: underline;">Mirarab, Siavash</a>
    
        <a href="https://orcid.org/0000-0001-5410-1518">
            <img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon" style="width:16px;height:16px;">
        </a>
    
    
    
         (<a href="../../affiliations/university-of-california-san-diego" style="text-decoration: underline;">University of California, San Diego</a>)
    
</div>
        
            <div class="added-meta" style="display: flex; align-items: center;">
    <a href="../../author/rabiee-maryam/" style="text-decoration: underline;">Rabiee, Maryam</a>
    
    
    
         (<a href="../../affiliations/university-of-california-san-diego" style="text-decoration: underline;">University of California, San Diego</a>)
    
</div>
        
    <div class="added-meta">
    Published: 2023 on Dryad<br />


    <a href="https://doi.org/10.6076/d1cp4r">
    <img src="https://img.shields.io/badge/DOI-10.6076/d1cp4r-blue" alt="DOI">
    </a>
    </div>

  <div class="post-content"><h1 id="abstract">Abstract<a hidden class="anchor" aria-hidden="true" href="#abstract">#</a></h1>
<p>Motivation: Phylogenomics faces a dilemma: on the one hand, the most accurate species and gene tree estimation methods are those that co-estimate them; on the other hand, these co-estimation methods do not scale to moderately large numbers of species. The summary-based methods, which first infer gene trees independently and then combine them, are much more scalable but are prone to gene tree estimation error, which is inevitable when inferring trees from limited-length data. Gene tree estimation error is not just random noise and can create biases such as long-branch attraction. Results: We introduce a scalable likelihood-based approach to co-estimation under the multi-species coalescent model. The method, called quartet co-estimation (QuCo), takes as input independently inferred distributions over gene trees and computes the most likely species tree topology and internal branch length for each quartet, marginalizing over gene tree topologies and ignoring branch lengths by making several simplifying assumptions. It then updates the gene tree posterior probabilities based on the species tree. The focus on gene tree topologies and the heuristic division to quartets enables fast likelihood calculations. We benchmark our method with extensive simulations for quartet trees in zones known to produce biased species trees and further with larger trees. We also run QuCo on a biological dataset of bees. Our results show better accuracy than the summary-based approach ASTRAL run on estimated gene trees. Availability and implementation: QuCo is available on <a href="https://github.com/maryamrabiee/quco">https://github.com/maryamrabiee/quco</a>. Supplementary information Supplementary data are available at Bioinformatics online.</p>
<h2 id="methods">Methods<a hidden class="anchor" aria-hidden="true" href="#methods">#</a></h2>
<p>Data are simulated by us and provided here for reproducibility.</p>
<h2 id="technicalinfo">TechnicalInfo<a hidden class="anchor" aria-hidden="true" href="#technicalinfo">#</a></h2>
<p>Quco Dataset &mdash; Date belonging to the following paper: Rabiee, Maryam, and Siavash Mirarab. “QuCo: Quartet-Based Co-Estimation of Species Trees and Gene Trees.” Bioinformatics 38, no. Supplement1 (June 24, 2022): i413–21. . ## Description of the data and file structure There are several files: ### <code>Quartet-simulation-sequences.tar.gz</code> The main simulations presented in the paper, which involve Felsenstein’s zone quartets. Here, we provide the simulated sequences. Files are of the form: <code>rep.[CU]d/R[long]l-[short]s/[rep]/seq[seqlength]/sequences.tar.gz</code> where * <code>[seqlength]</code> is the sequence length and is either 1600, 800, 400, 200 * <code>[rep]</code> is the replicate number, which is between 1 and 20 * <code>[short]</code> is the length of the short branch, and varies between 0.01, 0.02, 0.04, 0.08. * <code>[long]</code> is the length of the long branch and varies between 0.1, 0.2, 0.3, and 0.4. * <code>[CU]</code> is the length of the internal branch length in coalescent units (CU) and varies between 0.1, 0.2, and 0.3. Each file includes the simulated sequences in the fasta format for all genes of each repliate. ### <code>Anomaly-simulations-mrbayes-outputs.tar.gz</code> This includes the results of the anomaly zone simulations, and specifically the output of MrBayes. For each of the 50 replicated simulations, we include: * <code>[id]/mrbayes-outputs.tar.gz</code> Inside each archive, we have MrBayes MCMC sample from 200 loci. The files are named as follows, where <code>locus id</code> is the name of the locus and we have results of four chains (runs 1&ndash;4). Each .t file includes the MCMC samples in nexus format, as generated by MrBayes. * <code>seq600/[locus id]/[locus id].nex.run[1/2/3/4].t</code> ### <code>BiologicalDataset_rj_tree_distribution.tar.gz</code> Biological dataset of Bossert et al. (2021) with 32 species and 1291 UCEs (). Here, for reproducabililty, we provide MrBayes tree distributions. For each of the 1291 loci, you can find the following files: * <code>uce-[ucid].run[1/2].t</code>: The MCM sample in nexus format from MrBayes, for chain (run) 1 or 2. ## Sharing/Access information The rest of the data is available on * <a href="https://gitlab.com/mrabiee/quo-data">GitLab</a> ## Code/Software The simulated data are generated using Simphy</p>
<h1 id="access-points">Access Points<a hidden class="anchor" aria-hidden="true" href="#access-points">#</a></h1>
<p><a href="https://datadryad.org/stash/dataset/doi:10.6076/D1CP4R">https://datadryad.org/stash/dataset/doi:10.6076/D1CP4R</a></p>
<h1 id="related-identifiers">Related Identifiers<a hidden class="anchor" aria-hidden="true" href="#related-identifiers">#</a></h1>
<h2 id="iscitedby">IsCitedBy<a hidden class="anchor" aria-hidden="true" href="#iscitedby">#</a></h2>
<ul>
<li><a href="https://doi.org/bioinformatics/btac265">https://doi.org/bioinformatics/btac265</a></li>
</ul>
<h2 id="isderivedfrom">IsDerivedFrom<a hidden class="anchor" aria-hidden="true" href="#isderivedfrom">#</a></h2>
<ul>
<li><a href="https://github.com/maryamrabiee/quco">https://github.com/maryamrabiee/quco</a></li>
</ul>


  </div>

  
    
    
    
      
    
  

  <footer class="post-footer">
    <ul class="post-tags">
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/co-estimation-methods/">Co-Estimation Methods</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/phylogenomics/">Phylogenomics</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/bayesian-phylogenetics/">Bayesian phylogenetics</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/species-tree-inference/">species tree inference</a></li>
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://datapurifier.github.io/landingpage-in-a-box/">Landingpage-in-a-Box</a></span>
    <span>
        Powered by
        <a href="https://gohugo.io/" rel="noopener noreferrer" target="_blank">Hugo</a> &
        <a href="https://github.com/adityatelange/hugo-PaperMod/" rel="noopener" target="_blank">PaperMod</a>
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
