<!DOCTYPE html>
<html lang="en" dir="auto">

<head><meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>Data from: The role of mutation bias in adaptive molecular evolution: insights from convergent changes in protein function | Landingpage-in-a-Box</title>
<meta name="keywords" content="Zonotrichia capensis, Merganetta armata, Selasphorus platycercus, Pterophanes cyanopterus, Boissonneaua matthewsii, Periparus ater aemodius, Lophonetta specularioides alticola, Hydropsalis longirostris, Aglaeactis castelnaudii, Colibri coruscans, Aegithalos bonvaloti, Anas cyanoptera orinoma, Lophonetta specularioides specularioides, Eriocnemis luciani, High Altitude, Metriopelia melanoptera, Lophophanes dichrous, Chalcostigma stanleyi, Hemoglobin, Conirostrum cinereum, Poecile palustris, Schistes geoffroyi, map[FOS:Biological sciences], Cinclodes albiventris, Coeligena violifer, Parus minor, Oreotrochilus estella, Diglossa glauca, Chalcostigma ruficeps, Anas georgica, Coeligena coeligena, Troglodytes aedon, Anser anser, Catamenia analis, Heliodoxa leadbeateri, Cyanochen cyanoptera, Anairetes nigrocristatus, Anser indicus, Chloephaga melanoptera, Hydropsalis descussata, Pygochelidon cyanoleuca, Anas flavirostris oxyptera, Haplophaedia aureliae, Amazilia amazilia, Anas versicolor, Anas flavirostris flavirostris, Neochen jubata, protein evolution, Columbina cruziana, Anas puna, Pteronetta hartlaubi, Notiochelidon murina, Adelomyia melanogenys, Archilochus alexandri, Anairetes reguloides, Furnarius leucopus, Spinus magellanicus, Parus humilis, Diglossa brunneiventris, Periparus ater pekinensis, Tangara nigroviridis, Oxyura jamaicensis, Aegithalos fuliginosus, Anas cyanoptera cyanoptera, Amazilia viridicauda, CpG, Mutation bias, Tangara vassorii">
<meta name="description" content="Abstract An underexplored question in evolutionary genetics concerns the extent to which mutational bias in the production of genetic variation influences outcomes and pathways of adaptive molecular evolution. In the genomes of at least some vertebrate taxa, an important form of mutation bias involves changes at CpG dinucleotides: If the DNA nucleotide cytosine (C) is immediately 5’ to guanine (G) on the same coding strand, and if the C is methylated, then C→T and G→A mutations occur at an elevated rate relative to mutations at non-CpG sites.">
<meta name="author" content="Storz, Jay F., Natarajan, Chandrasekhar, Signore, Anthony V., Witt, Christopher C., McCandlish, David M., Stoltzfus, Arlin">
<link rel="canonical" href="https://datapurifier.github.io/landingpage-in-a-box/10.5061/dryad.2256f38/">
<link crossorigin="anonymous" href="/landingpage-in-a-box/assets/css/stylesheet.4652fda8867d1835187bccecf539b89d6bb1fa82083e7fc03287396443458ee0.css" integrity="sha256-RlL9qIZ9GDUYe8zs9Tm4nWux&#43;oIIPn/AMoc5ZENFjuA=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/landingpage-in-a-box/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://datapurifier.github.io/landingpage-in-a-box/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://datapurifier.github.io/landingpage-in-a-box/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://datapurifier.github.io/landingpage-in-a-box/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://datapurifier.github.io/landingpage-in-a-box/apple-touch-icon.png">
<link rel="mask-icon" href="https://datapurifier.github.io/landingpage-in-a-box/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="Data from: The role of mutation bias in adaptive molecular evolution: insights from convergent changes in protein function" />
<meta property="og:description" content="Abstract An underexplored question in evolutionary genetics concerns the extent to which mutational bias in the production of genetic variation influences outcomes and pathways of adaptive molecular evolution. In the genomes of at least some vertebrate taxa, an important form of mutation bias involves changes at CpG dinucleotides: If the DNA nucleotide cytosine (C) is immediately 5’ to guanine (G) on the same coding strand, and if the C is methylated, then C→T and G→A mutations occur at an elevated rate relative to mutations at non-CpG sites." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://datapurifier.github.io/landingpage-in-a-box/10.5061/dryad.2256f38/" /><meta property="article:section" content="10.5061" />



<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Data from: The role of mutation bias in adaptive molecular evolution: insights from convergent changes in protein function"/>
<meta name="twitter:description" content="Abstract An underexplored question in evolutionary genetics concerns the extent to which mutational bias in the production of genetic variation influences outcomes and pathways of adaptive molecular evolution. In the genomes of at least some vertebrate taxa, an important form of mutation bias involves changes at CpG dinucleotides: If the DNA nucleotide cytosine (C) is immediately 5’ to guanine (G) on the same coding strand, and if the C is methylated, then C→T and G→A mutations occur at an elevated rate relative to mutations at non-CpG sites."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [
    {
      "@type": "ListItem",
      "position":  1 ,
      "name": "10.5061",
      "item": "https://datapurifier.github.io/landingpage-in-a-box/10.5061/"
    }, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Data from: The role of mutation bias in adaptive molecular evolution: insights from convergent changes in protein function",
      "item": "https://datapurifier.github.io/landingpage-in-a-box/10.5061/dryad.2256f38/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "Data from: The role of mutation bias in adaptive molecular evolution: insights from convergent changes in protein function",
  "name": "Data from: The role of mutation bias in adaptive molecular evolution: insights from convergent changes in protein function",
  "description": "Abstract An underexplored question in evolutionary genetics concerns the extent to which mutational bias in the production of genetic variation influences outcomes and pathways of adaptive molecular evolution. In the genomes of at least some vertebrate taxa, an important form of mutation bias involves changes at CpG dinucleotides: If the DNA nucleotide cytosine (C) is immediately 5’ to guanine (G) on the same coding strand, and if the C is methylated, then C→T and G→A mutations occur at an elevated rate relative to mutations at non-CpG sites.",
  "keywords": [
    "Zonotrichia capensis", "Merganetta armata", "Selasphorus platycercus", "Pterophanes cyanopterus", "Boissonneaua matthewsii", "Periparus ater aemodius", "Lophonetta specularioides alticola", "Hydropsalis longirostris", "Aglaeactis castelnaudii", "Colibri coruscans", "Aegithalos bonvaloti", "Anas cyanoptera orinoma", "Lophonetta specularioides specularioides", "Eriocnemis luciani", "High Altitude", "Metriopelia melanoptera", "Lophophanes dichrous", "Chalcostigma stanleyi", "Hemoglobin", "Conirostrum cinereum", "Poecile palustris", "Schistes geoffroyi", {"FOS":"Biological sciences"}, "Cinclodes albiventris", "Coeligena violifer", "Parus minor", "Oreotrochilus estella", "Diglossa glauca", "Chalcostigma ruficeps", "Anas georgica", "Coeligena coeligena", "Troglodytes aedon", "Anser anser", "Catamenia analis", "Heliodoxa leadbeateri", "Cyanochen cyanoptera", "Anairetes nigrocristatus", "Anser indicus", "Chloephaga melanoptera", "Hydropsalis descussata", "Pygochelidon cyanoleuca", "Anas flavirostris oxyptera", "Haplophaedia aureliae", "Amazilia amazilia", "Anas versicolor", "Anas flavirostris flavirostris", "Neochen jubata", "protein evolution", "Columbina cruziana", "Anas puna", "Pteronetta hartlaubi", "Notiochelidon murina", "Adelomyia melanogenys", "Archilochus alexandri", "Anairetes reguloides", "Furnarius leucopus", "Spinus magellanicus", "Parus humilis", "Diglossa brunneiventris", "Periparus ater pekinensis", "Tangara nigroviridis", "Oxyura jamaicensis", "Aegithalos fuliginosus", "Anas cyanoptera cyanoptera", "Amazilia viridicauda", "CpG", "Mutation bias", "Tangara vassorii"
  ],
  "articleBody": "Abstract An underexplored question in evolutionary genetics concerns the extent to which mutational bias in the production of genetic variation influences outcomes and pathways of adaptive molecular evolution. In the genomes of at least some vertebrate taxa, an important form of mutation bias involves changes at CpG dinucleotides: If the DNA nucleotide cytosine (C) is immediately 5’ to guanine (G) on the same coding strand, and if the C is methylated, then C→T and G→A mutations occur at an elevated rate relative to mutations at non-CpG sites. Here we examine experimental data from case studies in which it has been possible to identify the causative substitutions that are responsible for adaptive changes in the functional properties of vertebrate hemoglobin (Hb). Specifically, we examine the molecular basis of convergent increases in Hb-O2 affinity in high-altitude birds. Using a data set of experimentally verified, affinity-enhancing mutations in the Hbs of highland avian taxa, we tested whether causative changes are enriched for mutations at CpG dinucleotides relative to the frequency of CpG mutations among all possible missense mutations. The tests revealed that a disproportionate number of causative amino acid replacements were attributable to CpG mutations, demonstrating that mutation bias can influence outcomes of molecular adaptation.\nTechnicalInfo README for supplementary data from Storz, et al. This is the main README file for the supplementary data package for the study of CpG bias in avian hemoglobin adaptation by Storz, et al., entitled “The role of mutation bias in adaptive molecular evolution: insights from convergent changes in protein function” (10.1098/rstb.2018.0238). # EXPLANATION Based on prior work, Storz, et al. provide a summary of experimentally validated changes in hemoglobin affinity associated with altitude adaptation in 35 pairs of closely related low- and high-altitude bird populations. The authors report on a statistically significant association of CpG mutation hotspots with these adaptive changes. This supplementary data package provides a script and data files that reproduce the main results by processing the primary data, recomputing the null expectations, and conducting statistical tests. The data also may be useful to researchers carrying out meta-analyses of cases in which adaptation can be traced to specific molecular changes. # DESCRIPTION OF FILES The files are all text files, including sequence alignment files in FASTA format, tables in csv (comma-separated values) format, and an R-MarkDown script: * altitude_pairs_changes_2018-09-13.csv (primary data file describing validated changes) * altitude_pairs_changes_meta.csv (data definition file for above) * hb_CpG_analysis.Rmd (script to process data, calculate null models, carry out tests) * alpha-globin_ancestral_seqs.fas (reconstructed ancestral Hb alpha A sequences) * beta-globin_ancestral_seqs.fas (reconstructed ancestral Hb beta sequences) * README.txt (this file) # HOW TO OBTAIN THE CONTENTS The contents of this package are available from Dryad () or from the authors (see TECHNICAL SUPPORT). # SOFTWARE REQUIREMENTS The files are all text files readable with a text editor. The csv (comma-separated values) files can be opened with Excel and a variety of other tools. The script is written in R-markdown and is best used in RStudio. The code chunks in the Rmd file also can be copied and pasted into an R environment. # VALIDATION The hb_CpG_analysis.Rmd script will generate script_output/altitude_pairs_obs_exp.csv (null expectations and counts) and script_output/test_results.csv (P values). The contents of these files can be compared with values reported in the manuscript. Use the following assertions to validate the integrity of the files and the correctness of processing without the Rmd script: * each FASTA file has exactly 35 ancestral CDS sequences, one for each pair * there are no gaps in the sequences * the main data file describes 10 paths with 6, 3, 3, 2, 2, 2, 1, 1, 1, 1 events * there are 6 G to S (Gly to Ser) changes at site 83 in HBB * the HBA change from A to V at site 63 is in a CpG context * the HBA change from P to A at site 119 is not in a CpG context # TECHNICAL SUPPORT For questions about data processing and statistical analysis, contact Arlin Stoltzfus (). For questions about experimental data and natural history of the bird species, contact Jay Storz (). # STATEMENT OF RIGHTS AND DISCLAIMER The authors release the data and scripts in this package into the public domain, with no warranty, under a Creative Commons Zero (CC0 1.0) license.\nAccess Points https://datadryad.org/stash/dataset/doi:10.5061/dryad.2256f38\nRelated Identifiers IsCitedBy https://doi.org/10.1098/rstb.2018.0238 IsDerivedFrom https://doi.org/10.5281/zenodo.10198318 ",
  "wordCount" : "722",
  "inLanguage": "en",
  "datePublished": "0001-01-01T00:00:00Z",
  "dateModified": "0001-01-01T00:00:00Z",
  "author":[{
    "@type": "Person",
    "name": "Storz, Jay F."
  }, {
    "@type": "Person",
    "name": "Natarajan, Chandrasekhar"
  }, {
    "@type": "Person",
    "name": "Signore, Anthony V."
  }, {
    "@type": "Person",
    "name": "Witt, Christopher C."
  }, {
    "@type": "Person",
    "name": "McCandlish, David M."
  }, {
    "@type": "Person",
    "name": "Stoltzfus, Arlin"
  }],
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://datapurifier.github.io/landingpage-in-a-box/10.5061/dryad.2256f38/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "Landingpage-in-a-Box",
    "logo": {
      "@type": "ImageObject",
      "url": "https://datapurifier.github.io/landingpage-in-a-box/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://datapurifier.github.io/landingpage-in-a-box/" accesskey="h" title="Landingpage-in-a-Box (Alt + H)">Landingpage-in-a-Box</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">

<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      Data from: The role of mutation bias in adaptive molecular evolution: insights from convergent changes in protein function
    </h1>
    <div class="post-meta">

</div>
  </header> 
    
    
      
        
            <div class="added-meta" style="display: flex; align-items: center;">
    Storz, Jay F.
    
    
    
         (University of Nebraska - Lincoln)
    
</div>
        
            <div class="added-meta" style="display: flex; align-items: center;">
    Natarajan, Chandrasekhar
    
    
    
         (University of Nebraska - Lincoln)
    
</div>
        
            <div class="added-meta" style="display: flex; align-items: center;">
    Signore, Anthony V.
    
    
    
         (University of Nebraska - Lincoln)
    
</div>
        
            <div class="added-meta" style="display: flex; align-items: center;">
    Witt, Christopher C.
    
    
    
         (University of New Mexico)
    
</div>
        
            <div class="added-meta" style="display: flex; align-items: center;">
    McCandlish, David M.
    
    
    
         (Cold Spring Harbor Laboratory)
    
</div>
        
            <div class="added-meta" style="display: flex; align-items: center;">
    Stoltzfus, Arlin
    
        <a href="https://orcid.org/0000-0002-0963-1357">
            <img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon" style="width:16px;height:16px;">
        </a>
    
    
    
         (National Institute of Standards and Technology)
    
</div>
        
    <div class="added-meta">
    Published: 2019 on Dryad<br />


    <a href="https://doi.org/10.5061/dryad.2256f38">
    <img src="https://img.shields.io/badge/DOI-10.5061/dryad.2256f38-blue" alt="DOI">
    </a>
    </div>

  <div class="post-content"><h1 id="abstract">Abstract<a hidden class="anchor" aria-hidden="true" href="#abstract">#</a></h1>
<p>An underexplored question in evolutionary genetics concerns the extent to which mutational bias in the production of genetic variation influences outcomes and pathways of adaptive molecular evolution. In the genomes of at least some vertebrate taxa, an important form of mutation bias involves changes at CpG dinucleotides: If the DNA nucleotide cytosine (C) is immediately 5’ to guanine (G) on the same coding strand, and if the C is methylated, then C→T and G→A mutations occur at an elevated rate relative to mutations at non-CpG sites. Here we examine experimental data from case studies in which it has been possible to identify the causative substitutions that are responsible for adaptive changes in the functional properties of vertebrate hemoglobin (Hb). Specifically, we examine the molecular basis of convergent increases in Hb-O2 affinity in high-altitude birds. Using a data set of experimentally verified, affinity-enhancing mutations in the Hbs of highland avian taxa, we tested whether causative changes are enriched for mutations at CpG dinucleotides relative to the frequency of CpG mutations among all possible missense mutations. The tests revealed that a disproportionate number of causative amino acid replacements were attributable to CpG mutations, demonstrating that mutation bias can influence outcomes of molecular adaptation.</p>
<h2 id="technicalinfo">TechnicalInfo<a hidden class="anchor" aria-hidden="true" href="#technicalinfo">#</a></h2>
<p>README for supplementary data from Storz, et al. This is the main README file for the supplementary data package for the study of CpG bias in avian hemoglobin adaptation by Storz, et al., entitled &ldquo;The role of mutation bias in adaptive molecular evolution: insights from convergent changes in protein function&rdquo; (10.1098/rstb.2018.0238). # EXPLANATION Based on prior work, Storz, et al. provide a summary of experimentally validated changes in hemoglobin affinity associated with altitude adaptation in 35 pairs of closely related low- and high-altitude bird populations. The authors report on a statistically significant association of CpG mutation hotspots with these adaptive changes. This supplementary data package provides a script and data files that reproduce the main results by processing the primary data, recomputing the null expectations, and conducting statistical tests. The data also may be useful to researchers carrying out meta-analyses of cases in which adaptation can be traced to specific molecular changes. # DESCRIPTION OF FILES The files are all text files, including sequence alignment files in FASTA format, tables in csv (comma-separated values) format, and an R-MarkDown script: * altitude_pairs_changes_2018-09-13.csv (primary data file describing validated changes) * altitude_pairs_changes_meta.csv (data definition file for above) * hb_CpG_analysis.Rmd (script to process data, calculate null models, carry out tests) * alpha-globin_ancestral_seqs.fas (reconstructed ancestral Hb alpha A sequences) * beta-globin_ancestral_seqs.fas (reconstructed ancestral Hb beta sequences) * README.txt (this file) # HOW TO OBTAIN THE CONTENTS The contents of this package are available from Dryad () or from the authors (see TECHNICAL SUPPORT). # SOFTWARE REQUIREMENTS The files are all text files readable with a text editor. The csv (comma-separated values) files can be opened with Excel and a variety of other tools. The script is written in R-markdown and is best used in RStudio. The code chunks in the Rmd file also can be copied and pasted into an R environment. # VALIDATION The hb_CpG_analysis.Rmd script will generate script_output/altitude_pairs_obs_exp.csv (null expectations and counts) and script_output/test_results.csv (P values). The contents of these files can be compared with values reported in the manuscript. Use the following assertions to validate the integrity of the files and the correctness of processing without the Rmd script: * each FASTA file has exactly 35 ancestral CDS sequences, one for each pair * there are no gaps in the sequences * the main data file describes 10 paths with 6, 3, 3, 2, 2, 2, 1, 1, 1, 1 events * there are 6 G to S (Gly to Ser) changes at site 83 in HBB * the HBA change from A to V at site 63 is in a CpG context * the HBA change from P to A at site 119 is not in a CpG context # TECHNICAL SUPPORT For questions about data processing and statistical analysis, contact Arlin Stoltzfus (). For questions about experimental data and natural history of the bird species, contact Jay Storz (). # STATEMENT OF RIGHTS AND DISCLAIMER The authors release the data and scripts in this package into the public domain, with no warranty, under a Creative Commons Zero (CC0 1.0) license.</p>
<h1 id="access-points">Access Points<a hidden class="anchor" aria-hidden="true" href="#access-points">#</a></h1>
<p><a href="https://datadryad.org/stash/dataset/doi:10.5061/dryad.2256f38">https://datadryad.org/stash/dataset/doi:10.5061/dryad.2256f38</a></p>
<h1 id="related-identifiers">Related Identifiers<a hidden class="anchor" aria-hidden="true" href="#related-identifiers">#</a></h1>
<h2 id="iscitedby">IsCitedBy<a hidden class="anchor" aria-hidden="true" href="#iscitedby">#</a></h2>
<ul>
<li><a href="https://doi.org/10.1098/rstb.2018.0238">https://doi.org/10.1098/rstb.2018.0238</a></li>
</ul>
<h2 id="isderivedfrom">IsDerivedFrom<a hidden class="anchor" aria-hidden="true" href="#isderivedfrom">#</a></h2>
<ul>
<li><a href="https://doi.org/10.5281/zenodo.10198318">https://doi.org/10.5281/zenodo.10198318</a></li>
</ul>


  </div>

  
    
    
    
      
    
  

  <footer class="post-footer">
    <ul class="post-tags">
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/zonotrichia-capensis/">Zonotrichia capensis</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/merganetta-armata/">Merganetta armata</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/selasphorus-platycercus/">Selasphorus platycercus</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/pterophanes-cyanopterus/">Pterophanes cyanopterus</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/boissonneaua-matthewsii/">Boissonneaua matthewsii</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/periparus-ater-aemodius/">Periparus ater aemodius</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/lophonetta-specularioides-alticola/">Lophonetta specularioides alticola</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/hydropsalis-longirostris/">Hydropsalis longirostris</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/aglaeactis-castelnaudii/">Aglaeactis castelnaudii</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/colibri-coruscans/">Colibri coruscans</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/aegithalos-bonvaloti/">Aegithalos bonvaloti</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/anas-cyanoptera-orinoma/">Anas cyanoptera orinoma</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/lophonetta-specularioides-specularioides/">Lophonetta specularioides specularioides</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/eriocnemis-luciani/">Eriocnemis luciani</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/high-altitude/">High Altitude</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/metriopelia-melanoptera/">Metriopelia melanoptera</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/lophophanes-dichrous/">Lophophanes dichrous</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/chalcostigma-stanleyi/">Chalcostigma stanleyi</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/hemoglobin/">Hemoglobin</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/conirostrum-cinereum/">Conirostrum cinereum</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/poecile-palustris/">Poecile palustris</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/schistes-geoffroyi/">Schistes geoffroyi</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/cinclodes-albiventris/">Cinclodes albiventris</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/coeligena-violifer/">Coeligena violifer</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/parus-minor/">Parus minor</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/oreotrochilus-estella/">Oreotrochilus estella</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/diglossa-glauca/">Diglossa glauca</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/chalcostigma-ruficeps/">Chalcostigma ruficeps</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/anas-georgica/">Anas georgica</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/coeligena-coeligena/">Coeligena coeligena</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/troglodytes-aedon/">Troglodytes aedon</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/anser-anser/">Anser anser</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/catamenia-analis/">Catamenia analis</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/heliodoxa-leadbeateri/">Heliodoxa leadbeateri</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/cyanochen-cyanoptera/">Cyanochen cyanoptera</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/anairetes-nigrocristatus/">Anairetes nigrocristatus</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/anser-indicus/">Anser indicus</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/chloephaga-melanoptera/">Chloephaga melanoptera</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/hydropsalis-descussata/">Hydropsalis descussata</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/pygochelidon-cyanoleuca/">Pygochelidon cyanoleuca</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/anas-flavirostris-oxyptera/">Anas flavirostris oxyptera</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/haplophaedia-aureliae/">Haplophaedia aureliae</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/amazilia-amazilia/">Amazilia amazilia</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/anas-versicolor/">Anas versicolor</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/anas-flavirostris-flavirostris/">Anas flavirostris flavirostris</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/neochen-jubata/">Neochen jubata</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/protein-evolution/">protein evolution</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/columbina-cruziana/">Columbina cruziana</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/anas-puna/">Anas puna</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/pteronetta-hartlaubi/">Pteronetta hartlaubi</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/notiochelidon-murina/">Notiochelidon murina</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/adelomyia-melanogenys/">Adelomyia melanogenys</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/archilochus-alexandri/">Archilochus alexandri</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/anairetes-reguloides/">Anairetes reguloides</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/furnarius-leucopus/">Furnarius leucopus</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/spinus-magellanicus/">Spinus magellanicus</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/parus-humilis/">Parus humilis</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/diglossa-brunneiventris/">Diglossa brunneiventris</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/periparus-ater-pekinensis/">Periparus ater pekinensis</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/tangara-nigroviridis/">Tangara nigroviridis</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/oxyura-jamaicensis/">Oxyura jamaicensis</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/aegithalos-fuliginosus/">Aegithalos fuliginosus</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/anas-cyanoptera-cyanoptera/">Anas cyanoptera cyanoptera</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/amazilia-viridicauda/">Amazilia viridicauda</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/cpg/">CpG</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/mutation-bias/">Mutation bias</a></li>
      <li><a href="https://datapurifier.github.io/landingpage-in-a-box/tags/tangara-vassorii/">Tangara vassorii</a></li>
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://datapurifier.github.io/landingpage-in-a-box/">Landingpage-in-a-Box</a></span>
    <span>
        Powered by
        <a href="https://gohugo.io/" rel="noopener noreferrer" target="_blank">Hugo</a> &
        <a href="https://github.com/adityatelange/hugo-PaperMod/" rel="noopener" target="_blank">PaperMod</a>
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
