<!DOCTYPE html>
<html lang="en" dir="auto">

<head><meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>Annotation data from: Utilizing a comparative approach to assess genome evolution during diploidization in Artemisia tridentata (Asteraceae), a keystone species of western North America | Landingpage-in-a-Box</title>
<meta name="keywords" content="map[FOS:Biological sciences]">
<meta name="description" content="Abstract Supporting dataset for genome assembly data found within NCBI BioProjects PRJNA1032953 (UTT2), PRJNA722258 (IDT2) and PRJNA795150 (IDT3-Reference Genome). Fasta assembly data used in the EDTA analysis to generate subsequent output files are available from the NCBI Genome database and raw sequence data are available from the NCBI SRA database. Each input fasta contains the nine pseudo-chromosomes described in Melton et al. 2022. Reads from each sample were mapped to the nine pseudo-chromosomes and used to call a consensus sequence.">
<meta name="author" content="Melton, Anthony, Novak, Stephen, Buerki, Sven">
<link rel="canonical" href="https://datapurifier.github.io/landingpage-in-a-box/10.7923/hysd-x388/">
<link crossorigin="anonymous" href="/landingpage-in-a-box/assets/css/stylesheet.5cfc680b1eeaeef9efbced92d46c2a9e876b72ee14fba85846afc4cff9e6e6f8.css" integrity="sha256-XPxoCx7q7vnvvO2S1Gwqnodrcu4U&#43;6hYRq/Ez/nm5vg=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/landingpage-in-a-box/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://datapurifier.github.io/landingpage-in-a-box/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://datapurifier.github.io/landingpage-in-a-box/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://datapurifier.github.io/landingpage-in-a-box/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://datapurifier.github.io/landingpage-in-a-box/apple-touch-icon.png">
<link rel="mask-icon" href="https://datapurifier.github.io/landingpage-in-a-box/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="Annotation data from: Utilizing a comparative approach to assess genome evolution during diploidization in Artemisia tridentata (Asteraceae), a keystone species of western North America" />
<meta property="og:description" content="Abstract Supporting dataset for genome assembly data found within NCBI BioProjects PRJNA1032953 (UTT2), PRJNA722258 (IDT2) and PRJNA795150 (IDT3-Reference Genome). Fasta assembly data used in the EDTA analysis to generate subsequent output files are available from the NCBI Genome database and raw sequence data are available from the NCBI SRA database. Each input fasta contains the nine pseudo-chromosomes described in Melton et al. 2022. Reads from each sample were mapped to the nine pseudo-chromosomes and used to call a consensus sequence." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://datapurifier.github.io/landingpage-in-a-box/10.7923/hysd-x388/" /><meta property="article:section" content="10.7923" />



<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Annotation data from: Utilizing a comparative approach to assess genome evolution during diploidization in Artemisia tridentata (Asteraceae), a keystone species of western North America"/>
<meta name="twitter:description" content="Abstract Supporting dataset for genome assembly data found within NCBI BioProjects PRJNA1032953 (UTT2), PRJNA722258 (IDT2) and PRJNA795150 (IDT3-Reference Genome). Fasta assembly data used in the EDTA analysis to generate subsequent output files are available from the NCBI Genome database and raw sequence data are available from the NCBI SRA database. Each input fasta contains the nine pseudo-chromosomes described in Melton et al. 2022. Reads from each sample were mapped to the nine pseudo-chromosomes and used to call a consensus sequence."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [
    {
      "@type": "ListItem",
      "position":  1 ,
      "name": "10.7923",
      "item": "https://datapurifier.github.io/landingpage-in-a-box/10.7923/"
    }, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Annotation data from: Utilizing a comparative approach to assess genome evolution during diploidization in Artemisia tridentata (Asteraceae), a keystone species of western North America",
      "item": "https://datapurifier.github.io/landingpage-in-a-box/10.7923/hysd-x388/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "Annotation data from: Utilizing a comparative approach to assess genome evolution during diploidization in Artemisia tridentata (Asteraceae), a keystone species of western North America",
  "name": "Annotation data from: Utilizing a comparative approach to assess genome evolution during diploidization in Artemisia tridentata (Asteraceae), a keystone species of western North America",
  "description": "Abstract Supporting dataset for genome assembly data found within NCBI BioProjects PRJNA1032953 (UTT2), PRJNA722258 (IDT2) and PRJNA795150 (IDT3-Reference Genome). Fasta assembly data used in the EDTA analysis to generate subsequent output files are available from the NCBI Genome database and raw sequence data are available from the NCBI SRA database. Each input fasta contains the nine pseudo-chromosomes described in Melton et al. 2022. Reads from each sample were mapped to the nine pseudo-chromosomes and used to call a consensus sequence.",
  "keywords": [
    {"FOS":"Biological sciences"}
  ],
  "articleBody": "Abstract Supporting dataset for genome assembly data found within NCBI BioProjects PRJNA1032953 (UTT2), PRJNA722258 (IDT2) and PRJNA795150 (IDT3-Reference Genome). Fasta assembly data used in the EDTA analysis to generate subsequent output files are available from the NCBI Genome database and raw sequence data are available from the NCBI SRA database. Each input fasta contains the nine pseudo-chromosomes described in Melton et al. 2022. Reads from each sample were mapped to the nine pseudo-chromosomes and used to call a consensus sequence. The EDTA analysis provides several outputs, listed below. The primary file of interest is the “SAMPLE_consensus.fasta.mod.EDTA.TEanno.gff3” file. This file was used as inputs for comparisons of TE content across the three genomes. Please visit https://github.com/oushujun/EDTA for more information about the EDTA pipeline.\nFILES: SAMPLE_consensus.fasta.mod.EDTA.TEanno.gff3 == Whole-genome TE annotation SAMPLE_consensus.fasta.mod.EDTA.TEanno.sum == Summary of whole-genome TE annotation SAMPLE_consensus.fasta.mod.EDTA.TElib.fa == A non-redundant TE library SAMPLE_consensus.fasta.mod.MAKER.masked == Low-threshold TE masking\nAccess Points https://data.nkn.uidaho.edu/dataset/annotation-data-utilizing-comparative-approach-assess-genome-evolution-during-diploidization\n",
  "wordCount" : "149",
  "inLanguage": "en",
  "datePublished": "0001-01-01T00:00:00Z",
  "dateModified": "0001-01-01T00:00:00Z",
  "author":[{
    "@type": "Person",
    "name": "Melton, Anthony"
  }, {
    "@type": "Person",
    "name": "Novak, Stephen"
  }, {
    "@type": "Person",
    "name": "Buerki, Sven"
  }],
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://datapurifier.github.io/landingpage-in-a-box/10.7923/hysd-x388/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "Landingpage-in-a-Box",
    "logo": {
      "@type": "ImageObject",
      "url": "https://datapurifier.github.io/landingpage-in-a-box/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://datapurifier.github.io/landingpage-in-a-box/" accesskey="h" title="Landingpage-in-a-Box (Alt + H)">Landingpage-in-a-Box</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">

<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      Annotation data from: Utilizing a comparative approach to assess genome evolution during diploidization in Artemisia tridentata (Asteraceae), a keystone species of western North America
    </h1>
    <div class="post-meta">

</div>
  </header> 
    
    
      
        
            <div class="post-meta" style="display: flex; align-items: center;">
    Melton, Anthony
    
        <a href="https://orcid.org/0000-0003-4882-6664">
            <img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon" style="width:16px;height:16px;">
        </a>
    
    
    
         (Boise State University)
    
</div>
        
            <div class="post-meta" style="display: flex; align-items: center;">
    Novak, Stephen
    
        <a href="https://orcid.org/0000-0002-0537-7953">
            <img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon" style="width:16px;height:16px;">
        </a>
    
    
    
         (Boise State University)
    
</div>
        
            <div class="post-meta" style="display: flex; align-items: center;">
    Buerki, Sven
    
        <a href="https://orcid.org/0000-0002-8299-6539">
            <img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon" style="width:16px;height:16px;">
        </a>
    
    
    
         (Boise State University)
    
</div>
        
    <div class="post-meta">
    Published: 2023 on University of Idaho


    <a href="https://doi.org/10.7923/hysd-x388">
    <img src="https://img.shields.io/badge/DOI-10.7923/hysd-x388-blue" alt="DOI">
    </a>
    </div>

      
    <div id="mapid" style="height: 180px;"></div>

    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css" />
    <script src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js"></script>

    <script>
    var mymap = L.map('mapid').setView([9.15125,-79.8553], 5);
    L.tileLayer('https://api.mapbox.com/styles/v1/{id}/tiles/{z}/{x}/{y}?access_token=pk.eyJ1Ijoic2t5YnJpc3RvbCIsImEiOiI1ZGI4NTAxOGIwNDJlY2U0YWU3ZTcxYjE0ZGQyNDRiZSJ9.B-iS3a18Fnw5ln5Aa5JdCw', {
        maxZoom: 18,
        attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/">OpenStreetMap</a> contributors, ' +
        '<a href="https://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, ' +
        'Imagery © <a href="https://www.mapbox.com/">Mapbox</a>',
        id: 'mapbox/streets-v11',
        tileSize: 512,
        zoomOffset: -1
    }).addTo(mymap);

    var arrfeatures = [];

    

    
        var bounds = [[38.306,-116.9641],[43.3371,-109.3876]];
        var rectangle = L.rectangle(bounds, {color: "#ff7800", weight: 1}).addTo(mymap);
        arrfeatures.push(rectangle);
        
        var group = new L.featureGroup([rectangle]);
        
        mymap.fitBounds(group.getBounds());
    
    </script>

  <div class="post-content"><h1 id="abstract">Abstract<a hidden class="anchor" aria-hidden="true" href="#abstract">#</a></h1>
<p>Supporting dataset for genome assembly data found within NCBI BioProjects PRJNA1032953 (UTT2), PRJNA722258 (IDT2) and PRJNA795150 (IDT3-Reference Genome). Fasta assembly data used in the EDTA analysis to generate subsequent output files are available from the NCBI Genome database and raw sequence data are available from the NCBI SRA database. Each input fasta contains the nine pseudo-chromosomes described in Melton et al. 2022. Reads from each sample were mapped to the nine pseudo-chromosomes and used to call a consensus sequence. The EDTA analysis provides several outputs, listed below. The primary file of interest is the &ldquo;SAMPLE_consensus.fasta.mod.EDTA.TEanno.gff3&rdquo; file. This file was used as inputs for comparisons of TE content across the three genomes.
Please visit <a href="https://github.com/oushujun/EDTA">https://github.com/oushujun/EDTA</a> for more information about the EDTA pipeline.</p>
<p>FILES:
SAMPLE_consensus.fasta.mod.EDTA.TEanno.gff3 == Whole-genome TE annotation
SAMPLE_consensus.fasta.mod.EDTA.TEanno.sum == Summary of whole-genome TE annotation
SAMPLE_consensus.fasta.mod.EDTA.TElib.fa == A non-redundant TE library
SAMPLE_consensus.fasta.mod.MAKER.masked == Low-threshold TE masking</p>
<h1 id="access-points">Access Points<a hidden class="anchor" aria-hidden="true" href="#access-points">#</a></h1>
<p><a href="https://data.nkn.uidaho.edu/dataset/annotation-data-utilizing-comparative-approach-assess-genome-evolution-during-diploidization">https://data.nkn.uidaho.edu/dataset/annotation-data-utilizing-comparative-approach-assess-genome-evolution-during-diploidization</a></p>


  </div>

  
    
    
    
      
        <h4>Contributors: DataManager</h2>
        
          <div class="post-meta" style="display: flex; align-items: center;">
    Child, Andrew Wright
    
        <a href="https://orcid.org/0000-0001-6666-0739">
            <img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon" style="width:16px;height:16px;">
        </a>
    
    
    
         (University of Idaho)
    
</div>
        
      
    
      
    
  

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://datapurifier.github.io/landingpage-in-a-box/">Landingpage-in-a-Box</a></span>
    <span>
        Powered by
        <a href="https://gohugo.io/" rel="noopener noreferrer" target="_blank">Hugo</a> &
        <a href="https://github.com/adityatelange/hugo-PaperMod/" rel="noopener" target="_blank">PaperMod</a>
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
