---
title: "Data for: A histone demethylase links the loss of plasticity to nongenetic inheritance and morphological change"
doi: 10.5061/dryad.98sf7m0nm
referralUrl: https://datadryad.org/stash/dataset/doi:10.5061/dryad.98sf7m0nm
categories:
- Dataset
tags:
- Geometric Morphometrics
- plasticity
- Gene expression
- FOS: Biological sciences
- Pristionchus
- episodic positive selection
- histone demethylase
publishers:
- Dryad
author:
- Levis, Nicholas
- Ragsdale, Erik
affiliations:
- Indiana University
funders:
- National Science Foundation
---

# Abstract
Plasticity is a widespread feature of development, enabling phenotypic change based on the environment. Although the evolutionary loss of plasticity has been linked both theoretically and empirically to increased rates of phenotypic diversification, molecular insights into how this process might unfold are generally lacking. Here, we show that a regulator of nongenetic inheritance links evolutionary loss of plasticity in nature to changes in plasticity and morphology as selected in the laboratory. Across nematodes of Diplogastridae, which ancestrally had a polyphenism, or discrete plasticity, in their feeding morphology, we use molecular evolutionary analyses to screen for change associated with independent losses of plasticity. Having inferred a set of ancestrally polyphenism-biased genes from phylogenetically informed gene-knockouts and gene-expression comparisons, selection signatures associated with plasticityâ€™s loss identify the histone H3K4 di/monodemethylase gene spr-5/LSD1/KDM1A. Manipulations of this gene affect both sensitivity and variation in plastic morphologies, and artificial selection of manipulated lines drives multigenerational shifts in these phenotypes. Our findings thus give mechanistic insight into how traits are modified as they traverse the continuum of greater to lesser environmental sensitivity.

## Methods
These data and R code correspond to-identification of morph-biased genes in Pristionchus fissidentatus (Pfi) constitutive mutants (files labeled MB),-results from aBSREL tests of episodic, diversifying selection on assimilated species (files labeled PS),-results from RELAX tests of weakened selection on assimilated species (files labeled RS),-phenotyping of mouth form in various experiments in Pfi and Pristionchus pacificus (Ppa) (files labeled PTYPE), and-geometric morphometrics of mouth form in spr-5 and wild-type lines (files labeled GM)-inferred potential effects of amino acid substitutions in Pristionchus bucculentus and Pristionchus elegans (IA)

## Other
R, Excel or similar spreadsheet software, text editor

## TechnicalInfo
A histone demethylase links the loss of plasticity to nongenetic inheritance and morphological change --- These data and R code correspond to -identification of morph-biased genes in Pristionchus fissidentatus (Pfi) constitutive mutants (files labeled MB), -results from aBSREL tests of episodic, diversifying selection on assimilated species (files labeled PS), -results from RELAX tests of weakened selection on assimilated species (files labeled RS), -phenotyping of mouth form in various experiments in Pfi and Pristionchus pacificus (Ppa) (files labeled PTYPE), and -geometric morphometrics of mouth form in spr-5 and wild-type lines (files labeled GM) -inferred potential effects of amino acid substitutions in Pristionchus bucculentus and Pristionchus elegans (IA) ## Description of the Data and file structure ### MB files * MB_Polyphenism.gene.lists.xlsx: Excel file with three tabs. Tabs one and two include the list of polyphenism genes identified for P. fissidentatus (Pfi) and P. pacificus. Columns indicate the morph a gene is associated with based on weighted gene co-expression network analysis (WGCNA_Morph) or differential expression analysis (Deseq_morph), whether these two match (N/A if only analysis is applicable) , a summary of the gene's match type (the same between analysis, disagreement between them, or only a single analysis applies), and the final morph bias assigned to the gene. If there was disagreement, the Deseq morph was chosen In columns, B and C, N/A indicates that gene was not present in that analysis result. The third tab includes the 'core' list of genes. Column D shows the best match of column B to the Ppa El Paco genome with V3 annotations. Column E is the assigned annotation based on Column D's best match to C. elegans. Useful for sorting and identifying candidate polyphenism associated genes. * MB_pfi_wgcna.R: R code for weighted gene co-expression network analysis of Pfi mutant gene expression data * MB_pfi.deseq2.R: R code for differential expression analysis of Pfi mutant gene expression data * MB_pfi.wgnca.geneInfo.csv: Comma-separated file summarizing module membership of Pfi genes. This file has a row for each gene and its morph association and module membership statistics for each module described in wgcna.R analysis * MB_pfi.mutants.counts.table.txt: Tab-separated file containing expression counts used as input for R scripts ### PS files PS_absrel.test.results.xlsx: Excel file with five tabs labeled for species contained therein. Columns indicate orthogroup to which a gene belongs, the gene name, the p-value, and the false discovery rate (fdr) corrected p-value. For individual species tabs, there is additionally the best match, if any, to the Pristionchus pacificus V3 genome annotation (PPA anno) and corresponding gene ID (Ppa gene) and morph bias of the gene (Poly morph) if any. These data might be useful for identifying genes based on morph association, gene name, or significance level. ### RS files RS_relax.test.results.xlsx: Excel file with five tabs labeled for contents: all results, significant results, significant relaxed results, P. bucculentus + P. elegans significant relaxed results, and D. magnus + L.texanum significant relaxed results. These data might be useful for identifying genes based on morph association, gene name, or significance level. Under all results, orthogroups in Dmag and Ltex are the same and Pel and Pbuc are the same, but those of Dmag and Ltex do not correpsond to the same groups as Pbuc and Pel. Columns denote the test statistic from HyPhy's likelihood ratio test (LRT), the corresponding p-value (P), the relaxation intensity parameter from Hyphy (Relax parameter), the orthogroup, species, false discovery rate corrected p-value (fdr), a representative gene from a given orthogroup of genes, the representativ egene's best match to Pristionchus pacificus V3 genome (if any) and the gene's morph bias, if any. ### PTYPE files * PTYPE_pfi.R: R code for assembling Pfi wild-type mouth form data into a dataframe and analyzing them * PTYPE_spr5.R: R code for assembling dataframes of mouth form data from various spr-5 experiments and analyzing the data * PTYPE_spr5.plasticity.R: R code for assembling dataframes of mouth form data from environmental induction experiments using spr-5 mutants * PTYPE_spr5.rs5200b.plasticity.R: R code for assembling dataframes of mouth form data from environmental induction experiments using spr-5 mutants in RS5200B background ### GM files * GM_spr5_initial_geomorph.rmd: R markdown file for initial analysis of spr-5 mutant and wild-type geometric morphometric data * GM_spr5_geomorph_AS.rmd: R markdown file file for analysis of spr-5 mutant geometic morphometric data following artificial selection * GM_NGIAS.lands_final.txt: Landmark data as input for GM_spr5_geomorph_AS.rmd file * GM_lands.final.txt: Landmark data as input for GM_spr5_initial_geomorph.rmd file * GM_spr5.double.mutant.rmd: R markdown file file for analysis of spr-5 mutant and eud-1 overexpression lines * GM_lands.double.txt: Landmark data as input for GM_lands.double.mutant.rmd file ### IA files * IA_inferred_ancestor_plus_Pbu.Pel.aln: Alignment of SPR-5 amino acid for P. bucculentus, P. elegans, and the inferred amino acid sequence for their shared node * IA_RAxML_bestTree_allPristionchus-rooted.tre: Tree file of the rooted, manually curated Pristionchus spr-5 gene tree * IA_spr5_functional_changes.xlsx: Excel file summarizing potential effects on SPR-5 function of sites likely experiencing and not experiencing positive selection in P. bucculentus and P.elegans. Amino acid positions used for the Phyre2 input and from the inferred ancestor are noted and aligned against each other and SPR-5 protein domains in the 'Phyre to inferred position' tab. This tab also contains mutational sensitivity (Mut. sens), site conservation (Conservation), presnece of catalytic activity, presence of pockets, and interface sites (PI site). The second and third tabs indicated the position, the ancestral amino acid at that position, the amino acid of the focal species, and whether or not that site had evidence of positive selection. These tabs further show the number of sites with and without positive selection evidence in each of the SPR-5 protein domains.

# Access Points
https://datadryad.org/stash/dataset/doi:10.5061/dryad.98sf7m0nm